// generate-datafaker/index.js
const { v4 } = require("uuid");

const bairros = [
  "Centro",
  "Rio Abaixo",
  "Jorge Teixeira",
  "Bom Parto",
  "Estação",
  "Setor Sônia Regina (Taquaralto)",
  "Alberto Maia",
  "Parque Nova Friburgo B",
  "Montanhão",
  "Lamarão",
  "Redenção",
  "Distrito Industrial III",
  "Cidade de Deus",
  "Boa Esperança",
  "Santa Maria",
  "Cidade Nova",
  "João de Deus",
  "Planalto Boa Esperança",
  "Vila Moreninha II",
  "Provedor I",
  "Loteamento Residencial Dom Osório Stoffel",
  "Vila das Flores",
  "Nova Teixeira",
  "Perpétuo Socorro",
  "Jardim União",
  "Asa Norte",
  "Mariana",
  "Mimoso",
  "Santo André",
  "Jardim das Palmeiras",
  "Mecejana",
  "Canafístula",
  "Dezoito do Forte",
  "Jardim Iracema",
  "Bangu",
  "Estância Velha",
  "Provedor I",
  "Mathias Velho",
  "Primavera",
  "União II",
  "Setor 04",
  "Boa Vista",
  "Parque do Bosque",
  "Jardim São Conrado",
  "Distrito Industrial",
  "Angelim",
  "Maria Leite",
  "Parque Perequê (Mambucaba)",
  "Canto da Varzea",
  "São Pedro",
  "Tabuleiro do Martins",
  "Raimundo Melo",
  "Hípica",
  "Pantanal",
  "Santa Maria",
  "Guaratiba",
  "Expedicionários",
  "Jorge Teixeira",
  "Jardim Veraneio",
  "Ceilândia Norte (Ceilândia)",
  "São Gerardo",
  "Barra da Tijuca",
  "Jardim das Américas",
  "Vila Bom Jesus",
  "Alto da Bondade",
  "Plano Diretor Sul",
  "Jardim Shangri-la A",
  "Góes Calmon",
  "Paroquial",
  "Cabralzinho",
  "Vila Mariana",
  "Chácaras Quintas Green Hill",
  "Ponte Alta (Gama)",
  "Gopoúva",
  "Nova Marabá",
  "Senador Hélio Campos",
  "Parque Eldorado",
  "Aeroporto",
  "Fazenda Vila Real de Itu",
  "Novo Esteio",
  "CPA IV",
  "Industrial",
  "Fonseca",
  "Industrial",
  "São Jorge",
  "Setor Santa Inês",
  "Eldorado",
  "Planalto Ayrton Senna",
  "Jardim Tropical",
  "Inácio Barbosa",
  "Setor Norte",
  "Jardim Presidencial",
  "Portal da Alegria",
  "Jardim Santa Júlia",
  "Redenção",
  "Centro",
  "Teixeira Leite",
  "São Francisco",
  "Boa Saúde",
  "Centro",
  "Sítio Cercado",
  "Mata Escura",
  "Vila Norte",
  "Conjunto Palmeiras",
  "Jóquei Clube",
  "25 de Julho",
  "Vila Alpina",
  "Mar do Norte",
  "Igapó",
  "São Clemente",
  "Timbí",
  "Vila Embratel",
  "Tinguazinho",
  "Setor Habitacional Arniqueira (Águas Claras)",
  "Sobradinho",
  "Canaã II",
  "Jardim dos Migrantes",
  "Loteamento Lunabel 3",
  "Infraero",
  "Centro",
  "Santa Mônica",
  "Jacintinho",
  "Grageru",
  "Vila Alegrete",
  "Mecejana",
  "Centro",
  "Roçado",
  "Espírito Santo",
  "Cinturão Verde",
  "Sucurijuquara (Mosqueiro)",
  "Jaderlândia",
  "Atalaia",
  "Mansões de Recreio Estrela D'alva VII",
  "Setor Sul (Taquaralto)",
  "Industrial",
  "Nossa Senhora Aparecida",
  "Parque Amazonas",
  "Promorar Vila Vitória",
  "Água Boa (Outeiro)",
  "Santa Rosa",
  "Santa Maria",
  "Nova Marabá",
  "Nordeste",
  "Setor Habitacional Jardim Botânico (Lago Sul)",
  "Serra Dourada",
  "Jardim Paquetá I",
  "Sitio",
  "Conjunto Procon",
  "Centro",
  "Alto Boqueirão",
  "Calhau",
  "Parque do Lago",
  "Plano Diretor Sul",
  "Sarandi",
  "Km 32",
  "Monte Castelo",
  "Nova Marabá",
  "São Sebastião",
  "Taquarussu",
  "Antares",
  "Riacho Doce",
  "Setor 02",
  "Ganchinho",
  "Conselheiro Alberto Silva",
  "Laranjeiras",
  "Capão do Pequi",
  "Santo Antônio",
  "Angelim",
  "Vila Mira",
  "Conceição",
  "Maria das Graças",
  "Nova Marília",
  "Bomba",
  "Riachuelo",
  "Betânia",
  "Morada de Santa Fé",
  "Cidade Universitária",
  "São Vicente",
  "Nova União 01",
  "Setor Habitacional Samambaia (Vicente Pires)",
  "Itapoã II",
  "Samambaia Norte (Samambaia)",
  "Olímpico",
  "Rio Branco",
  "Araxá",
  "Fazendinha",
  "Residencial Brasília",
  "Jóquei Clube",
  "Infraero",
  "Jurunas",
  "Caimbé",
  "Loteamento Cruzeiro",
  "Jatobá",
  "Vera Cruz",
  "Universitário",
  "Cidade da Esperança",
  "Jardim Maracanã",
  "Jardim Veraneio",
  "Julião Ramos",
  "Aeroporto",
  "São Martinho",
  "Tancredo Neves",
  "São Manuel",
  "Jardim Paraíso",
  "Ininga",
  "Industrial",
  "Centro",
  "Flores",
  "Agulha (Icoaraci)",
  "Presidente Médici",
  "Muruci (Fazendinha)",
  "Barreiro",
  "Nossa Senhora Aparecida",
  "Parangaba",
  "Vila Costa e Silva",
  "Conjunto Habitacional Nelson Marcondes",
  "Guaxindiba",
  "Vila Eduardo",
  "Santo Antônio",
  "Oitis",
  "Parque Residencial Rita Vieira",
  "Felipe Camarão",
  "Miguel Couto",
  "Funcionários",
  "Vila Amazonas",
  "São Vicente",
  "Loteamento Vila Matilde",
  "Nova Parnamirim",
  "Ponte dos Carvalhos",
  "Sobral",
  "Caçari",
  "Boa Vista",
  "Coqueiro",
  "Nossa Senhora da Apresentação",
  "Getúlio Vargas",
  "Vila Santa Luzia",
  "Setor Bueno",
  "Setor de Habitações Individuais Norte",
  "Ponta Negra",
  "Centro",
  "Urussanguinha",
  "Salto Weissbach",
  "Piçarreira",
  "Asa Norte",
  "Cidade Alta",
  "Olaria",
  "Metrópole",
  "Jardim Floresta",
  "Jacareacanga",
  "COHAB Anil I",
  "Boa Vista",
  "Tapanã (Icoaraci)",
  "Vale Quem Tem",
  "Cruzeiro",
  "Jardim Marco Zero",
  "Jaguaribe",
  "Distrito Industrial",
  "Vila Embratel",
  "Quinta Samambaia",
  "Boné Azul",
  "Samambaia Sul (Samambaia)",
  "Alexandre Zachia",
  "Pernambués",
  "Santa Edwiges",
  "Aeroclube",
  "Centro",
  "Vida Nova",
  "Ampliação",
  "Jardim Atlântico",
  "Santos Dumont I",
  "Alto do Peru",
  "Mecejana",
  "Novo Visconde",
  "Vila Paulista",
  "Setor Noroeste",
  "Margem Esquerda do Rio Negro",
  "Jardim Janaína (Taquaralto)",
  "Santo Amaro",
  "Lagoa da Conceição",
  "Parque Leblon",
  "Universidade",
  "Alto José Leal",
  "Morada do Sol",
  "Angelim",
  "Boné Azul",
  "São Marcos",
  "Alzira Ramos",
  "Cristo Redentor",
  "Chácara Ipê",
  "Plano Diretor Sul",
  "José Conrado de Araújo",
  "Mirueira",
  "Vila Jardim Pompéia",
  "Boa Vista",
  "Aeroporto",
  "São Francisco",
  "Santo Antônio",
  "Bernardo Monteiro",
  "Jardim do Cedro",
  "Mafrense",
  "Privê Residencial Elza Fronza",
  "Itapoã",
  "Gapara",
  "Nossa Senhora das Graças",
  "Centro",
  "Cidade Nova",
  "Petrópolis",
  "Benfica",
  "Aeroporto",
  "Jardim Independente I",
  "Santa Lúcia",
  "Centro",
  "São João",
  "Engenheiro Luciano Cavalcante",
  "Loteamento Joafra",
  "Tabajaras",
  "São Caetano",
  "Potengi",
  "Santo Antônio",
  "Ceilândia Sul (Ceilândia)",
  "Novo Horizonte",
  "Flores",
  "Capibaribe",
  "Nossa Senhora da Penha",
  "Ponto Novo",
  "Tancredo Neves",
  "Jardim Pinhal",
  "Centro",
  "Jardim Oceania",
  "Centro",
  "Coroa do Meio",
  "Boa Vista",
  "Jardim Uirapuru",
  "Centro",
  "Tabatinga",
  "Lourival Parente",
  "Garcia",
];
function generateNeighborhood() {
  return bairros[getRandomInt(0, bairros.length - 1)];
}

const ceps = [
  "37002190",
  "08694105",
  "69088210",
  "57017231",
  "58807292",
  "77060650",
  "54771600",
  "72887252",
  "09783440",
  "49088080",
  "29032781",
  "38044780",
  "69099035",
  "29312462",
  "72503300",
  "67140682",
  "56316675",
  "58065127",
  "79065208",
  "68927167",
  "78715868",
  "32605400",
  "45994314",
  "68905512",
  "78118808",
  "70711900",
  "76813644",
  "47803580",
  "29144849",
  "77817420",
  "69304445",
  "57302696",
  "49072293",
  "58807045",
  "21843701",
  "92031063",
  "68927113",
  "92340310",
  "85050030",
  "76913271",
  "76873480",
  "55038788",
  "65914508",
  "79093721",
  "68929568",
  "64040255",
  "79310170",
  "23953086",
  "64600170",
  "88351402",
  "57060175",
  "69921002",
  "91755022",
  "68907306",
  "99070345",
  "23035050",
  "58041006",
  "69088459",
  "79037087",
  "72275164",
  "60320050",
  "22621040",
  "81530380",
  "18072831",
  "53170250",
  "77021080",
  "86071981",
  "45605412",
  "64601030",
  "68906814",
  "74938050",
  "35056002",
  "72426125",
  "07092130",
  "68507090",
  "69316472",
  "25240430",
  "69310018",
  "13312370",
  "93270060",
  "78058148",
  "49066290",
  "24120200",
  "32220070",
  "79645060",
  "75526015",
  "69902410",
  "60766710",
  "69314717",
  "49041092",
  "73751182",
  "76901046",
  "64037440",
  "12228304",
  "69049330",
  "67030045",
  "29310263",
  "69305260",
  "93347440",
  "86010180",
  "81910020",
  "41225720",
  "77825796",
  "60870355",
  "69313128",
  "89290027",
  "32655274",
  "28898108",
  "59104291",
  "66643802",
  "54768580",
  "65081678",
  "26080255",
  "71995345",
  "73015312",
  "79843168",
  "76900663",
  "72862527",
  "68908864",
  "76963724",
  "88035125",
  "57040665",
  "49025420",
  "79900290",
  "69304525",
  "35963971",
  "88108250",
  "45037750",
  "69312366",
  "66933050",
  "68627504",
  "49035440",
  "72859661",
  "77064662",
  "85053516",
  "38400680",
  "65030720",
  "88806843",
  "66843780",
  "55028280",
  "72545522",
  "68511360",
  "59042145",
  "71680354",
  "78056220",
  "73755012",
  "58500975",
  "69918640",
  "64200270",
  "81770190",
  "65071520",
  "78120792",
  "77015570",
  "91150010",
  "26298578",
  "58101072",
  "68510107",
  "78098243",
  "77089980",
  "57048052",
  "57039550",
  "76873186",
  "81935381",
  "64209220",
  "38410240",
  "78134214",
  "49060650",
  "64034170",
  "04377160",
  "44067426",
  "29705087",
  "25902064",
  "64601424",
  "76913725",
  "30570710",
  "29143618",
  "57072450",
  "69303385",
  "76875680",
  "72001205",
  "35702380",
  "72329214",
  "69313265",
  "29147713",
  "68903887",
  "68911011",
  "72866230",
  "69313058",
  "68908821",
  "66026272",
  "69312188",
  "77817320",
  "45066356",
  "29146709",
  "29024510",
  "59070090",
  "02839030",
  "79037022",
  "68908150",
  "59607340",
  "88708704",
  "97032210",
  "26315150",
  "68458120",
  "64049865",
  "49066190",
  "65650970",
  "69028145",
  "66811070",
  "58417455",
  "68911395",
  "48070380",
  "69306005",
  "60710500",
  "13081110",
  "19813535",
  "24722301",
  "56328165",
  "89218218",
  "32141110",
  "79052550",
  "59074100",
  "26070371",
  "58079720",
  "68926090",
  "69303490",
  "37706604",
  "59151902",
  "54580498",
  "69912004",
  "69307760",
  "84073410",
  "66823069",
  "59114034",
  "49055310",
  "79116600",
  "74223165",
  "71536034",
  "59090510",
  "79904716",
  "88905402",
  "89032130",
  "64055605",
  "70857520",
  "59025764",
  "76801302",
  "26216090",
  "69312035",
  "69152252",
  "65050370",
  "93230642",
  "66831053",
  "64057465",
  "58415453",
  "68903189",
  "58015440",
  "64027575",
  "65081496",
  "12295710",
  "68909463",
  "72315017",
  "97503000",
  "41100470",
  "57310380",
  "58036811",
  "29010610",
  "59147045",
  "24808168",
  "53140090",
  "35022530",
  "40349630",
  "69304350",
  "27940318",
  "68552713",
  "70683305",
  "69001019",
  "77062186",
  "50100340",
  "88062235",
  "61631200",
  "68903640",
  "55608055",
  "69402564",
  "64040195",
  "68909136",
  "41250243",
  "29141107",
  "58070483",
  "69917558",
  "77015128",
  "49085010",
  "53405030",
  "74685740",
  "89012260",
  "49037310",
  "69063022",
  "64032250",
  "32010735",
  "94197510",
  "64004620",
  "74692002",
  "74940110",
  "65083514",
  "76804151",
  "09910970",
  "69094660",
  "69079363",
  "20911400",
  "69310118",
  "68372750",
  "57082382",
  "13900425",
  "90550020",
  "60811253",
  "69919392",
  "64067030",
  "40390690",
  "59120700",
  "58701020",
  "72235801",
  "58704745",
  "69058808",
  "54705182",
  "56912395",
  "49097140",
  "76829490",
  "07120030",
  "63138974",
  "58037245",
  "64558970",
  "49035640",
  "57303320",
  "68374110",
  "58832970",
  "53605500",
  "64022600",
  "89020302",
  ,
];

function generateCEP() {
  return ceps[getRandomInt(0, ceps.length - 1)];
}
const city = [
  "Varginha",
  "Suzano",
  "Manaus",
  "Maceió",
  "Sousa",
  "Palmas",
  "Camaragibe",
  "Cidade Ocidental",
  "São Bernardo do Campo",
  "Aracaju",
  "Vitória",
  "Uberaba",
  "Manaus",
  "Cachoeiro de Itapemirim",
  "Brasília",
  "Ananindeua",
  "Petrolina",
  "João Pessoa",
  "Campo Grande",
  "Santana",
  "Rondonópolis",
  "Betim",
  "Teixeira de Freitas",
  "Macapá",
  "Várzea Grande",
  "Brasília",
  "Porto Velho",
  "Barreiras",
  "Cariacica",
  "Araguaína",
  "Boa Vista",
  "Arapiraca",
  "Aracaju",
  "Sousa",
  "Rio de Janeiro",
  "Canoas",
  "Santana",
  "Canoas",
  "Guarapuava",
  "Ji-Paraná",
  "Ariquemes",
  "Caruaru",
  "Imperatriz",
  "Campo Grande",
  "Santana",
  "Teresina",
  "Corumbá",
  "Angra dos Reis",
  "Picos",
  "Brusque",
  "Maceió",
  "Rio Branco",
  "Porto Alegre",
  "Macapá",
  "Passo Fundo",
  "Rio de Janeiro",
  "João Pessoa",
  "Manaus",
  "Campo Grande",
  "Brasília",
  "Fortaleza",
  "Rio de Janeiro",
  "Curitiba",
  "Sorocaba",
  "Olinda",
  "Palmas",
  "Londrina",
  "Itabuna",
  "Picos",
  "Macapá",
  "Aparecida de Goiânia",
  "Governador Valadares",
  "Brasília",
  "Guarulhos",
  "Marabá",
  "Boa Vista",
  "Duque de Caxias",
  "Boa Vista",
  "Itu",
  "Esteio",
  "Cuiabá",
  "Aracaju",
  "Niterói",
  "Contagem",
  "Três Lagoas",
  "Itumbiara",
  "Rio Branco",
  "Fortaleza",
  "Boa Vista",
  "Aracaju",
  "Planaltina",
  "Ji-Paraná",
  "Teresina",
  "São José dos Campos",
  "Manaus",
  "Ananindeua",
  "Cachoeiro de Itapemirim",
  "Boa Vista",
  "Novo Hamburgo",
  "Londrina",
  "Curitiba",
  "Salvador",
  "Araguaína",
  "Fortaleza",
  "Boa Vista",
  "São Bento do Sul",
  "Betim",
  "Rio das Ostras",
  "Natal",
  "Belém",
  "Camaragibe",
  "São Luís",
  "Nova Iguaçu",
  "Brasília",
  "Brasília",
  "Dourados",
  "Ji-Paraná",
  "Novo Gama",
  "Macapá",
  "Cacoal",
  "Florianópolis",
  "Maceió",
  "Aracaju",
  "Ponta Porã",
  "Boa Vista",
  "Barra Feliz",
  "São José",
  "Vitória da Conquista",
  "Boa Vista",
  "Belém",
  "Paragominas",
  "Aracaju",
  "Luziânia",
  "Palmas",
  "Guarapuava",
  "Uberlândia",
  "São Luís",
  "Criciúma",
  "Belém",
  "Caruaru",
  "Brasília",
  "Marabá",
  "Natal",
  "Brasília",
  "Cuiabá",
  "Planaltina",
  "Monteiro",
  "Rio Branco",
  "Parnaíba",
  "Curitiba",
  "São Luís",
  "Várzea Grande",
  "Palmas",
  "Porto Alegre",
  "Nova Iguaçu",
  "Cabedelo",
  "Marabá",
  "Cuiabá",
  "Palmas",
  "Maceió",
  "Maceió",
  "Ariquemes",
  "Curitiba",
  "Parnaíba",
  "Uberlândia",
  "Várzea Grande",
  "Aracaju",
  "Teresina",
  "São Paulo",
  "Feira de Santana",
  "Colatina",
  "Magé",
  "Picos",
  "Ji-Paraná",
  "Belo Horizonte",
  "Cariacica",
  "Maceió",
  "Boa Vista",
  "Ariquemes",
  "Brasília",
  "Sete Lagoas",
  "Brasília",
  "Boa Vista",
  "Cariacica",
  "Macapá",
  "Macapá",
  "Novo Gama",
  "Boa Vista",
  "Macapá",
  "Belém",
  "Boa Vista",
  "Araguaína",
  "Vitória da Conquista",
  "Cariacica",
  "Vitória",
  "Natal",
  "São Paulo",
  "Campo Grande",
  "Macapá",
  "Mossoró",
  "Tubarão",
  "Santa Maria",
  "Queimados",
  "Tucuruí",
  "Teresina",
  "Aracaju",
  "São Francisco do Maranhão",
  "Manaus",
  "Belém",
  "Campina Grande",
  "Macapá",
  "Alagoinhas",
  "Boa Vista",
  "Fortaleza",
  "Campinas",
  "Assis",
  "São Gonçalo",
  "Petrolina",
  "Joinville",
  "Contagem",
  "Campo Grande",
  "Natal",
  "Nova Iguaçu",
  "João Pessoa",
  "Santana",
  "Boa Vista",
  "Poços de Caldas",
  "Parnamirim",
  "Cabo de Santo Agostinho",
  "Rio Branco",
  "Boa Vista",
  "Ponta Grossa",
  "Belém",
  "Natal",
  "Aracaju",
  "Campo Grande",
  "Goiânia",
  "Brasília",
  "Natal",
  "Ponta Porã",
  "Araranguá",
  "Blumenau",
  "Teresina",
  "Brasília",
  "Natal",
  "Porto Velho",
  "Nova Iguaçu",
  "Boa Vista",
  "Parintins",
  "São Luís",
  "Sapucaia do Sul",
  "Belém",
  "Teresina",
  "Campina Grande",
  "Macapá",
  "João Pessoa",
  "Teresina",
  "São Luís",
  "Caçapava",
  "Macapá",
  "Brasília",
  "Uruguaiana",
  "Salvador",
  "Arapiraca",
  "João Pessoa",
  "Vitória",
  "Parnamirim",
  "Itaboraí",
  "Olinda",
  "Governador Valadares",
  "Salvador",
  "Boa Vista",
  "Macaé",
  "Redenção",
  "Brasília",
  "Manaus",
  "Palmas",
  "Recife",
  "Florianópolis",
  "Caucaia",
  "Macapá",
  "Vitória de Santo Antão",
  "Manacapuru",
  "Teresina",
  "Macapá",
  "Salvador",
  "Cariacica",
  "João Pessoa",
  "Rio Branco",
  "Palmas",
  "Aracaju",
  "Paulista",
  "Goiânia",
  "Blumenau",
  "Aracaju",
  "Manaus",
  "Teresina",
  "Contagem",
  "Gravataí",
  "Teresina",
  "Goiânia",
  "Aparecida de Goiânia",
  "São Luís",
  "Porto Velho",
  "Diadema",
  "Manaus",
  "Manaus",
  "Rio de Janeiro",
  "Boa Vista",
  "Altamira",
  "Maceió",
  "Amparo",
  "Porto Alegre",
  "Fortaleza",
  "Rio Branco",
  "Teresina",
  "Salvador",
  "Natal",
  "Patos",
  "Brasília",
  "Patos",
  "Manaus",
  "São Lourenço da Mata",
  "Serra Talhada",
  "Aracaju",
  "Porto Velho",
  "Guarulhos",
  "Ponta da Serra",
  "João Pessoa",
  "São Miguel do Fidalgo",
  "Aracaju",
  "Arapiraca",
  "Altamira",
  "Mato Grosso",
  "Igarassu",
  "Teresina",
  "Blumenau",
];
function generateCity() {
  return city[getRandomInt(0, city.length - 1)];
}
const cnpj = [
  "08425199000101",
  "44135307000175",
  "93465006000115",
  "70058235000110",
  "98087360000177",
  "10890296000117",
  "23773058000173",
  "53517219000129",
  "35986793000144",
  "48000755000112",
  "55595775000149",
  "95877653000160",
  "03437713000105",
  "97367920000184",
  "02708629000116",
  "44007878000124",
  "42676005000189",
  "36464158000160",
  "44688403000140",
  "74733627000198",
  "17458255000103",
  "08271433000193",
  "95326402000197",
  "89041693000174",
  "43146037000135",
  "20963023000109",
  "44333495000146",
  "53003217000111",
  "92957192000147",
  "27348837000182",
  "75641096000176",
  "25941060000130",
  "24891986000103",
  "73830699000190",
  "66782110000160",
  "90716711000150",
  "57820167000116",
  "38787110000118",
  "62440667000162",
  "97133867000157",
  "51305920000159",
  "34216376000150",
  "14580760000120",
  "14898361000101",
  "80591820000134",
  "73400464000169",
  "84740307000119",
  "19365457000128",
  "30778779000131",
  "54123764000101",
  "29681985000140",
  "58631444000105",
  "52834766000175",
  "42871318000198",
  "75055429000185",
  "52968061000140",
  "84358351000169",
  "50725919000110",
  "46528689000122",
  "83955962000121",
  "04672960000159",
  "06369798000120",
  "36821956000100",
  "73850941000198",
  "53332792000168",
  "52328729000195",
  "29123710000190",
  "94902725000119",
  "95282969000109",
  "14115632000105",
  "04319522000101",
  "75212129000162",
  "33659004000135",
  "89271897000100",
  "44747123000166",
  "91558550000186",
  "41418832000100",
  "32936343000159",
  "41488205000145",
  "78897335000141",
  "45850444000154",
  "73832989000173",
  "44103667000195",
  "82398934000198",
  "60107996000160",
  "09732168000166",
  "13932650000117",
  "96798834000163",
  "03761687000176",
  "00620884000187",
  "30829694000135",
  "37409613000197",
  "54264640000138",
  "80037277000128",
  "65566252000128",
  "39450136000139",
  "73221596000123",
  "96603752000116",
  "31131607000134",
  "91625391000195",
  "86751642000148",
  "12472894000100",
  "46000351000101",
  "88903663000167",
  "63715986000104",
  "28936959000152",
  "15654489000192",
  "50574348000160",
  "68927455000190",
  "62435891000166",
  "32949275000162",
  "01063315000140",
  "23418719000142",
  "64088110000130",
  "27918637000118",
  "99962087000145",
  "24181732000193",
  "61234216000106",
  "17038273000128",
  "54123677000146",
  "28838193000173",
  "00792861000150",
  "55875299000110",
  "03131235000100",
  "15470029000104",
  "53966740000143",
  "60669322000150",
  "86593672000173",
  "05616956000136",
  "10290577000139",
  "80625620000155",
  "89159549000137",
  "99120878000128",
  "25479818000160",
  "70068936000130",
  "78302971000183",
  "62903416000177",
  "42748815000101",
  "25852582000165",
  "72540017000142",
  "86040159000155",
  "56364303000148",
  "86567828000141",
  "66828268000124",
  "03496345000176",
  "35472463000130",
  "98830880000128",
  "27104830000115",
  "82592694000168",
  "44376111000172",
  "43670584000115",
  "38292802000195",
  "45567488000171",
  "83833453000126",
  "68979094000126",
  "35611951000181",
  "80684083000114",
  "58628229000155",
  "12936147000177",
  "62653477000123",
  "47504953000150",
  "90271523000165",
  "27319813000103",
  "68598034000163",
  "40515489000159",
  "58146050000161",
  "74217829000187",
  "82222142000168",
  "51669301000100",
  "49367745000186",
  "09729777000182",
  "79242087000106",
  "78888896000104",
  ,
];
function generateCNPJ() {
  return cnpj[getRandomInt(0, cnpj.length - 1)];
}

const street = [
  "Rua Alves e Silva",
  "Rua Geny Gusmão dos Santos",
  "Rua dos Girassóis",
  "1ª Travessa Senador Rui Palmeira",
  "Rua Basílio Silva",
  "Rua das Margaridas",
  "Rua São Pedro",
  "Quadra Quadra 22",
  "Rua 18 de Agosto",
  "Rua Sete",
  "Beco Águia Dourada",
  "Rua Furnas",
  "Rua Emaús",
  "Rua Nazira Ginaíd Felipe",
  "Quadra EQ 203/303",
  "Travessa We-73",
  "Rua Sessenta e Dois",
  "Rua Operário João Joel de Souza",
  "Travessa Sambaiba",
  "Avenida 10",
  "Rua Rio Tapajós",
  "Rua Vinte",
  "Rua da Fé",
  "Avenida Nossa Senhora de Fátima",
  "Rua Palestina",
  "SCN Quadra 1 Bloco A",
  "Rua Onze de Outubro",
  "Rua Renascer",
  "Rua Quatro",
  "Rua C",
  "Travessa Araribóia",
  "Rua José Nino Lopes",
  "Travessa General Joaquim Inácio",
  "Rua Janduir Morais",
  "Rua Coronel Tamarindo",
  "Rua Trinta e Dois",
  "Travessa Ana Maria Coelho",
  "Rua Natal",
  "Rua Bartolomeu Bueno da Silva",
  "Rua do Sol",
  "Rua Papoulas",
  "Avenida Lourinaldo Soares da Silva",
  "Rua dos Cravos",
  "Rua Rio dos Cedros",
  "Avenida São João Apóstolo",
  "Rua Santa Zita",
  "Rua Rubi",
  "Rua Quarenta e Seis",
  "Avenida Severo Eulálio",
  "Rua São Pedro",
  "Travessa Rio da Silva",
  "Rua 25 de Dezembro",
  "Rua Senador Mondin",
  "Avenida Antônio Serafim dos Santos",
  "Travessa Santa Catarina",
  "Avenida Nilo Peçanha",
  "Avenida Presidente Epitácio Pessoa",
  "Rua Lauro Corona",
  "Rua Rosa Cavalheiro de Lima",
  "Quadra CNR 1 Conjunto G",
  "Travessa Hospedaria",
  "Rua General Ivan Raposo",
  "Rua Frei Francisco Sampaio",
  "Rua Jonas Leite",
  "Rua Joaquim de Freitas",
  "Quadra 308 Sul Alameda 7",
  "Avenida Tiradentes 1241 Bloco C Loja 6",
  "Avenida Aziz Maron",
  "Rua Parnaíba",
  "Avenida Norte Sul",
  "Rua dos Cariocas",
  "Rua A",
  "Rua Primavera",
  "Rua João Benedito Carlos",
  "Quadra Dez",
  "Rua HC-09",
  "Rua Acre",
  "Alameda SD-03",
  "Rua Inhambús",
  "Rua Pedro Carneiro Pereira",
  "Avenida Pica Pau Amarelo",
  "Rua Antônio Xavier",
  "Travessa Renato Pacheco Marques",
  "Rua Garibalde",
  "Rua Michel Thomé",
  "Praça Martins Correia",
  "Estrada São do Francisco",
  "Avenida Radialista João Ramos",
  "Rua CJ-11",
  "Rua H",
  "Quadra 11 MR 6",
  "Rua Vista Alegre",
  "Rua Rosa Tajra Adad",
  "Rua Francisco Pires de Moraes",
  "Rua Doutor Leonor Brilhante",
  "Rua Cavalcante",
  "Rua Projetada Oito",
  "Rua Alferes Paulo Saldanha",
  "Rua Ernesto Nicolau Brand",
  "Rua Mato Grosso",
  "Rua Campina Grande do Sul",
  "Rua Direta do Campo",
  "Rua Arapoema",
  "Rua Margarida Alves",
  "Rua Zircône",
  "Rua Estevão Buschle",
  "Rua Livramento",
  "Largo do Estácio",
  "3ª Travessa Santa Luzia",
  "Rua Vinte e Nove de Abril",
  "Rua dos Agrimensores",
  "Rua São Pantaleão",
  "Rua Nova",
  "Conjunto SHA Conjunto 5 Chácara 94",
  "Quadra Quadra 2 Conjunto C-12",
  "Rua Olindina Pereira da Silva",
  "Rua Padre Cícero",
  "Quadra Quadra 27",
  "Avenida Adolfo de Menezes",
  "Avenida Belo Horizonte",
  "Travessa Vilmar José de Souza",
  "Travessa Luiz Rêgo Barros",
  "Rua Maestro Domício Fraga",
  "Rua Nilo Barém",
  "Travessa João Antony",
  "Rua Principal, s/n",
  "Rua Camilo Veríssimo da Silva",
  "Rua TG-27",
  "Rua José Moreno",
  "Loteamento Olinda III",
  "Rua Princesa Isabel",
  "Praça José Alves",
  "Quadra Quadra 137",
  "Rua NC 15",
  "Rua Nelson Bastos Virmond",
  "Rua Veríssimo",
  "Rua Humaitá",
  "Rua Tereza Floriano Angélica",
  "Travessa São João",
  "Rua Pedro Bergamo",
  "Quadra QR 315 Área Especial C",
  "Quadra Dois",
  "Vila São João",
  "Condomínio Residencial Taboquinha",
  "Rua das Violetas",
  "Quadra Quadra 12",
  "Sítio Pitombeira, s/n",
  "Rua Flor Lírio Lustosa",
  "Rua Francisco Correia",
  "Rua Luiz Carlos Muggiati",
  "Rua Trinta e Dois",
  "Travessa Antônio Gonçalves Sobrinho",
  "Quadra 403 Sul Alameda 8",
  "Avenida Francisco Silveira Bitencourt",
  "Rua E",
  "Rua Severino Laurentino Leite",
  "Rua D",
  "Rua M",
  "Rua 20 10 Quadra 70 Lote 13",
  "Rua Maria Luzia Macedo da Silva",
  "Rua G",
  "Rua Jandaias",
  "Rua Irmão Luiz Albano",
  "Rua José Pinheiro",
  "Rua Rolande Bruno",
  "Rua Jatobá",
  "Praça Siqueira de Menezes",
  "Rua Professora Elenir Costa Lago",
  "Rua Capuavinha",
  "Rua Lajedo",
  "Rua Salvador Pireda",
  "Rua Dom Alfredo",
  "Rua Osildo Albano",
  "Rua Júlio Prestes",
  "Avenida Teresa Cristina",
  "Rua Rio Guaíba",
  "Rua Professora Maria Luíza Gusmão de Moraes",
  "Travessa Araraquara",
  "Avenida Perimetral Leste",
  "Rua Rua 4C Chácara 16",
  "Rua José Pereira Pinto",
  "Quadra QR 433 Conjunto 10",
  "Avenida Antonia Corrêa Lira",
  "Rua Moreira Cabral",
  "Travessa D",
  "Travessa Quinta do Matadouro",
  "Quadra Quadra 26",
  "Rua Silver",
  "Rua Agostinho Nogueira de Souza",
  "Vila Laura",
  "Rua Pastor Fernando Granjeiro",
  "Rua das Camélias",
  "Rua Trinta e Nove",
  "Rua Sabiá",
  "Rua Flamboyans",
  "Rua José Américo",
  "Rua da Poesia Épica",
  "Rua Mossoró",
  "Rua José Serafim",
  "Rua Felipe Camarão",
  "Rua Maria S. Linhares",
  "Rua Geraldo Balconi",
  "Rua Sêneca",
  "Rua D",
  "Rua Eduardo de Castro Neiva",
  "Rua Três",
  "Avenida Policarpo Freire, s/n",
  "Rua Uruguai",
  "Passagem José Bonifácio",
  "Rua Doutor Francisco Brasileiro",
  "Avenida Hubelino Nunes de Almeida",
  "Travessa Murilo Cavalcante",
  "Avenida Brigadeiro Eduardo Gomes",
  "Rua Polônia",
  "Rua dos Iguás",
  "Rua Antônio Rocha",
  "Rua Resende da Costa",
  "Rua Nelson Moura",
  "Rua Petrolândia",
  "Rua Vinte e Quatro",
  "Rua Macário Alves",
  "Rua Professor Antônio Trigueiro",
  "Rua Doutor Daniel de Melo",
  "Rua Manoel Luiz dos Santos",
  "Avenida Lucena de Azevedo",
  "Avenida Terêncio Lima",
  "Rua João Cioffi",
  "Avenida Ayrton Senna 2023",
  "Rua B",
  "Rua 3 de Agosto",
  "Rua Maria Coêlho",
  "Rua General Barros Falcão",
  "Alameda Nove",
  "Travessa Moura",
  "Rua Geruzinho",
  "Rua Santa Bernadete",
  "Rua T 60",
  "Trecho Trecho 02 Chácara 106",
  "Rua Manoel Lelé",
  "Avenida Brasil",
  "Rua Armelindo Cesa",
  "Rua das Acácias",
  "Rua Juiz Deolindo Júnior",
  "Quadra CLN 409 Bloco B",
  "Travessa Beira Canal",
  "Avenida Farquar",
  "Rua José Moacir Nogueira",
  "Rua Detson Mendes",
  "Travessa Um",
  "Rua Barão de Tromai",
  "Rua Otacílio C Duarte",
  "Alameda Cinquenta e Três",
  "Rua José Porfírio Gomes",
  "Rua Beltrando de Farias Castro",
  "Rua Pastor Deocleciano Cabralzinho de Assis",
  "Rua Isabel Pereira dos Santos Justino",
  "Rua São Lázaro",
  "Rua Jatobás",
  "Rua Ametista",
  "Rua Jatoba",
  "Quadra QR 513 Conjunto 17",
  "Rua Quinze de Novembro",
  "Rua Santa Maria",
  "Rua Luíza Maria de Olinda",
  "Praça Tenente Antonio Fernandes do Amaral",
  "Praça Pedro Caetano",
  "Rua da Fraternidade",
  "Rua Francisca Jorge de Souza",
  "Rua Neusta Pierre",
  "Rua Onze",
  "Travessa São José",
  "Avenida Mário Homem de Melo",
  "Rua Francelina Rocha",
  "Rua Wartello Prudente",
  "Quadra SQNW 103 Bloco K",
  "Margem Esquerda do Rio Negro",
  "Rua J8",
  "Rua Israel Fonseca",
  "Servidão Ailton João Peres",
  "Rua Filgueiras Sampaio",
  "Rua Francisca Feitosa dos Santos",
  "Rua Posto Médico",
  "Rua 4",
  "Rua São Francisco de Assis",
  "Avenida Gregório de Matos",
  "1ª Travessa São Paulo",
  "Rua Um",
  "Rua João Cavalcante de Albuquerque",
  "Alameda das Acácias",
  "Quadra AV 107 Sul Avenida NS 5",
  "Rua Adelaide de Souza Ferraz",
  "Rua Riacho Fragosinho",
  "Rua Monte Castelo",
  "Rua João Gonçalves",
  "Rua João Batista Pimentel",
  "Beco Mossoró",
  "Rua Maria José dos Santos",
  "Travessa Onze",
  "Rua John Lennon",
  "Rua Inácio Costa",
  "Rua Cora Coralina",
  "Rua 9A",
  "Rua Valdir de Aquino",
  "Rua Almirante Barroso",
  "Avenida Nossa Senhora das Vitórias 199/203",
  "Rua 83",
  "Rua Coronel Matos da Silveira",
  "Rua Ademar Serpa",
  "Rua Yeyê Coelho",
  "Rua Isaac Benarroch",
  "Rua Carteiro José Florentino",
  "Praça Paul P. Harris",
  "Rua Dezesseis de Julho",
  "Rua Blumenau",
  "Rua da Amizade",
  "Rua Lageada",
  "Rua da Cobal",
  "Rua Ilhéus",
  "Rua Dom Pedro II",
  "Quadra QNP 28 Conjunto A",
  "Rua Aluízio Queiroz",
  "Rua Desembargador Gaspar Guimarães",
  "2ª Travessa Imperial",
  "Rua Projetada Francisco de Souza Melo",
  "Rua Coronel José Menezes Filho",
  "Rua Florestan Fernandes",
  "Rua Haroldo Lobo",
  "Praça da Matriz 10",
  "Rua Huerta Ferreira de Melo",
  "Rua Joaquim Dias de Oliveira, s/n",
  "Rua Péricles Vieira de Azevedo",
  "Rua Manoel Saturnino de Almeida",
  "Rua Uberaba",
  "Rua Antonio Pereira Nunes, s/n",
  "Rua Mnaoel Rosendo de Lima",
  "Loteamento Santa Rita",
  "Rua Araranguá",
];

function generateStreet() {
  return street[getRandomInt(0, street.length - 1)];
}
const state = [
  "MG",
  "SP",
  "AM",
  "AL",
  "PB",
  "TO",
  "PE",
  "GO",
  "ES",
  "DF",
  "MS",
  "AP",
  "MT",
  "BA",
  "RO",
  "RR",
  "SE",
  "RJ",
  "RS",
  "MA",
  "PI",
  "SC",
  "AC",
  "CE",
  "RN",
];
function generateState() {
  return state[getRandomInt(0, state.length - 1)];
}

const cpfs = [
  "89579706069",
  "47982754090",
  "59129969050",
  "10850847010",
  "88992169086",
  "60235724050",
  "62434236030",
  "46459767025",
  "55284076052",
  "77577097098",
  "72536396002",
  "49978445005",
  "61355954070",
  "53441912004",
  "87060681035",
  "48505095090",
  "27593477029",
  "30960543040",
  "66511338029",
  "12202741003",
  "53239659085",
  "47886456025",
  "86467735072",
  "31724686070",
  "79493062031",
  "73634731028",
  "82820007015",
  "40736402080",
  "79903636075",
  "57595547008",
  "99552515068",
  "40441245072",
  "89364506030",
  "99847878005",
  "86216449087",
  "41934197033",
  "64440161006",
  "96283874053",
  "53295717079",
  "86229861011",
  "52038460060",
  "75557509020",
  "99329477097",
  "49624188076",
  "18057294035",
  "49940446004",
  "92385281015",
  "31772670065",
  "13161690060",
  "53334571076",
  "81030430012",
  "65214057059",
  "36027303018",
  "89126850079",
  "43663338053",
  "41662064004",
  "06259219091",
  "40792890060",
  "96536332016",
  "83472373016",
  "78066304064",
  "13734579023",
  "91131050002",
  "27675476000",
  "13148265068",
  "19539520037",
  "39343111053",
  "66902258015",
  "07906049063",
  "48408200070",
  "66608371080",
  "04053443091",
  "85820062094",
  "73455146090",
  "79146233008",
  "09937568080",
  "12558939076",
  "23455849040",
  "29006105082",
  "92126917061",
  "36869768030",
  "24822588041",
  "13178998063",
  "26165786030",
  "14120535010",
  "57570959035",
  "43423108053",
  "79261630073",
  "05681107060",
  "66743353043",
  "68702149052",
  "75575119050",
  "85526814041",
  "09884905010",
  "77055826004",
  "14715955016",
  "08091418094",
  "76334893084",
  "02170905081",
  "57084433082",
  "58270015032",
  "37883579088",
  "16916165081",
  "26273951063",
  "69992625007",
  "16431385020",
  "78755580009",
  "02594321052",
  "10039842070",
  "88099388011",
  "03569155064",
  "95933418030",
  "44746541027",
  "44021636099",
  "81700530054",
  "18428377022",
  "34529924033",
  "57869545015",
  "64925506099",
  "77946456034",
  "63566360090",
  "63981682084",
  "74503362003",
  "97523325017",
  "45223070046",
  "14798729035",
  "69679957071",
  "16512060075",
  "14787470051",
  "96799033036",
  "92549648080",
  "83354825006",
  "49521957085",
  "83268287000",
  "76394102075",
  "80428265081",
  "41100199071",
  "77821784010",
  "53812795078",
  "48557195010",
  "08588197081",
  "99131935087",
  "28694100002",
  "76585012046",
  "58345988032",
  "55656549046",
  "82656383021",
  "45850347062",
  "04290259049",
  "63564567011",
  "51906215022",
  "89992447095",
  "54012115030",
  "08187516062",
  "30018035094",
  "70654472009",
  "39087701063",
  "20218389000",
  "45900702006",
  "85514930009",
  "01551992000",
  "32995592065",
  "99949471095",
  "92762185050",
  "02693278007",
  "91576525031",
  "11767093055",
  "16674287059",
  "73815746086",
  "23526672083",
  "24611164071",
  "00213831031",
  "10282284060",
  "78128105000",
  "87455303009",
  "97412993038",
  "96799447000",
  "74951137070",
  "59778573069",
  "59622338089",
  "94594053041",
  "66830916097",
  "67413676087",
  "48359340080",
  "71642476099",
  "73401407007",
  "70980188059",
  "06410109075",
  "10700446010",
  "26418301080",
  "50582715083",
  "88490632022",
  "03843941084",
  "37140072027",
  "10127926097",
  "69953437009",
  "97627271002",
  "08189880020",
  "09498801088",
  "56045946014",
  "46450710098",
  "89882740081",
  "24969928070",
  "95297927005",
  "85762647064",
  "78712767000",
  "08512582049",
  "64277330010",
  "19482568001",
  "61602360081",
  "17429309054",
  "80350090017",
  "99786073070",
  "64886385052",
  "18719824050",
  "29517487010",
  "04843190004",
  "55386862057",
  "32774399092",
  "85066570080",
  "37909857080",
  "96267473074",
  "05561633031",
  "71371601070",
  "81453885030",
  "88757128081",
  "61303724006",
  "74698854040",
  "80559429002",
  "12189053013",
  "79560312065",
  "70492482071",
  "90658103075",
  "15789223052",
  "23263051083",
  "79910805014",
  "85013873045",
  "66605288068",
  "55468221047",
  "32421333075",
  "20036325066",
  "82721560085",
  "24324254052",
  "67082291027",
  "86134478059",
  "53244099017",
  "72577348010",
  "99327767004",
  "78990535018",
  "51436278007",
  "22524034038",
  "37745776090",
  "59010802035",
  "94109983010",
  "45180148049",
  "70289218080",
  "40301250006",
  "62823142088",
  "70684748029",
  "32242776029",
  "46875964015",
  "37137103090",
  "65468588005",
  "54074464055",
  "41244022012",
  "74596710023",
  "23391821078",
  "28759025042",
  "83912063001",
  "76534078031",
  "01674527039",
  "55088980000",
  "64977083083",
  "66817373077",
  "52362151026",
  "76455164024",
  "94344989023",
  "30229831079",
  "70480776032",
  "48710957030",
  "38352678094",
  "51754152040",
  "23133204050",
  "75640783087",
  "27204832060",
  "25576205057",
  "77005919033",
  "05518425040",
  "88362146060",
  "49617977079",
  "84179784025",
  "19098783082",
  "72953861068",
  ,
];
function generateCPF() {
  return cpfs[getRandomInt(0, cpfs.length - 1)];
}

const emails = [
  "seguranca@rafaeleraulfinanceirame.com.br",
  "desenvolvimento@sophieeclaudiacorretoresassociadosltda.com.br",
  "faleconosco@beatrizeleonardoentregasexpressasme.com.br",
  "diretoria@suelieluanpizzariame.com.br",
  "tesouraria@elisaegeraldoinformaticame.com.br",
  "treinamento@iagoemanuellimpezaltda.com.br",
  "contato@claudioebenjaminconsultoriafinanceiraltda.com.br",
  "auditoria@giovanaejoaolavanderialtda.com.br",
  "representantes@joaoecauepizzarialtda.com.br",
  "fabricacao@ninaecauatelasltda.com.br",
  "auditoria@gabrielaeamandaesportesme.com.br",
  "suporte@vanessaealiciapizzariame.com.br",
  "treinamento@renanecamilaalimentosme.com.br",
  "contabil@gustavoeluziaconsultoriafinanceiraltda.com.br",
  "contato@stefanyeeliascorretoresassociadosme.com.br",
  "faleconosco@luizaeenzomudancasltda.com.br",
  "ti@allanaethomastelecomunicacoesltda.com.br",
  "atendimento@jennifereandremarcenariame.com.br",
  "administracao@agathaeemilyentregasexpressasltda.com.br",
  "rh@ritaeisaactelasltda.com.br",
  "suporte@louiseeenricotelecomltda.com.br",
  "treinamento@emanueleruanpaesedocesme.com.br",
  "qualidade@sergioehelenaferragensme.com.br",
  "estoque@igoreisistelasme.com.br",
  "seguranca@luanaecarlalocacoesdeautomoveisme.com.br",
  "producao@caioeluccapaesedocesltda.com.br",
  "financeiro@leonardoepedrohenriquecontabilme.com.br",
  "atendimento@agathaeleandrobuffetme.com.br",
  "almoxarifado@fatimaetaniapublicidadeepropagandame.com.br",
  "representantes@luanepriscilapadariame.com.br",
  "ti@bentoegabrielfilmagensme.com.br",
  "seguranca@renanedanielacasanoturname.com.br",
  "producao@saraelevirestauranteltda.com.br",
  "seguranca@vitorejoanaconstrucoesme.com.br",
  "ouvidoria@gabrielaemarinaconsultoriafinanceiraltda.com.br",
  "representantes@marcelaehenriquevidrosltda.com.br",
  "auditoria@deboraelucianadocessalgadosme.com.br",
  "representantes@raulemanuelacomerciodebebidasme.com.br",
  "ouvidoria@marioefilipepizzariadeliveryltda.com.br",
  "producao@veraefabiopaesedocesme.com.br",
  "ouvidoria@juliaecarlosdocessalgadosme.com.br",
  "juridico@sophieeluciadocessalgadosltda.com.br",
  "almoxarifado@joseemanoelconstrucoesme.com.br",
  "fabricacao@isisethiagolavanderiame.com.br",
  "ouvidoria@marianeeelianetransportesme.com.br",
  "producao@antoniaehadassagraficame.com.br",
  "contato@patriciaerenanconsultoriafinanceiraltda.com.br",
  "producao@nelsonejulioentregasexpressasme.com.br",
  "vendas@lunaeenzolocacoesdeautomoveisme.com.br",
  "representantes@luccaenoahmarcenariame.com.br",
  "auditoria@alanaeluciajoalheriame.com.br",
  "administracao@teresinhaesophiatelasltda.com.br",
  "ouvidoria@maluecaioconstrucoesme.com.br",
  "manutencao@fernandaealicialimpezame.com.br",
  "pesquisa@isabellaeiagoassessoriajuridicame.com.br",
  "financeiro@priscilaeandreadocessalgadosltda.com.br",
  "cobranca@silvanaeviniciuspizzariadeliveryme.com.br",
  "cobranca@maiteeaurorainformaticame.com.br",
  "estoque@henriqueesophiafinanceiraltda.com.br",
  "estoque@eduardaejoseentulhosme.com.br",
  "treinamento@luziaemarcosviniciusadvocacialtda.com.br",
  "comunicacoes@mateuserayssafinanceiraltda.com.br",
  "sac@sueliethiagolocacoesdeautomoveisme.com.br",
  "presidencia@eloaepauloadegame.com.br",
  "compras@antoniaethiagofinanceiraltda.com.br",
  "diretoria@simoneeemilypizzariadeliveryltda.com.br",
  "cobranca@alessandraebenjamingraficaltda.com.br",
  "marketing@renatoeaylatelecomunicacoesme.com.br",
  "producao@diegoeviniciustelecomunicacoesltda.com.br",
  "seguranca@edsoneliviamarcenariame.com.br",
  "producao@pietroemartinadegaltda.com.br",
  "auditoria@thaleseauroracasanoturname.com.br",
  "ti@rayssaenicolecomerciodebebidasltda.com.br",
  "posvenda@carolinaeveramudancasltda.com.br",
  "estoque@carlaealiceconstrucoesltda.com.br",
  "manutencao@eduardoegiovannifotografiasme.com.br",
  "cobranca@manuelaenelsontelecomunicacoesme.com.br",
  "cobranca@mariaheclaricevidrosltda.com.br",
  "representantes@robertoecauepaesedocesltda.com.br",
  "desenvolvimento@marcelaelaisentulhosltda.com.br",
  "qualidade@veraemanoelmarketingltda.com.br",
  "qualidade@mariahefilipeentregasexpressasme.com.br",
  "orcamento@arthurenelsonadvocaciame.com.br",
  "comunicacoes@alexandreemariahpaesedocesltda.com.br",
  "faleconosco@elisaemarcosassessoriajuridicaltda.com.br",
  "fabricacao@bentoeevelynbuffetltda.com.br",
  "pesquisa@helenaeleonardomarketingltda.com.br",
  "financeiro@josefaecarolinaconstrucoesme.com.br",
  "posvenda@pedrohenriqueeemanuellyassessoriajuridicaltda.com.br",
  "rh@marcoseviniciuscomerciodebebidasltda.com.br",
  "estoque@lucasenicolemudancasltda.com.br",
  "contabilidade@taniaeyuriassessoriajuridicame.com.br",
  "treinamento@ianepedroadvocacialtda.com.br",
  "qualidade@heloiseemarianalavanderiame.com.br",
  "rh@felipeebeneditavidrosme.com.br",
  "contato@eduardoemarleneferragensltda.com.br",
  "presidencia@lunaeiagodocessalgadosme.com.br",
  "fiscal@otavioesimoneassessoriajuridicame.com.br",
  "faleconosco@antonioetanialavanderialtda.com.br",
  "seguranca@lorenaeluistelecomltda.com.br",
  "fabricacao@beneditaeigoralimentosltda.com.br",
  "contato@migueleallanalimpezame.com.br",
  "administracao@carolineegaelfilmagensltda.com.br",
  "faleconosco@jessicaerenancontabilltda.com.br",
  "producao@gabriellyejoanajoalheriame.com.br",
  "atendimento@joaquimesamuelcontabilme.com.br",
  "representantes@alexandreegiovanatelecomltda.com.br",
  "desenvolvimento@marianeebarbarabuffetme.com.br",
  "suporte@esteresophiacomerciodebebidasme.com.br",
  "manutencao@gabrielaemarcelaadvocaciame.com.br",
  "fabricacao@thiagoeelisaalimentosltda.com.br",
  "estoque@marciaecalebemudancasltda.com.br",
  "representantes@robertoenicolepizzariadeliveryme.com.br",
  "auditoria@raulecarlaferragensltda.com.br",
  "financeiro@arthureoliverdocessalgadosltda.com.br",
  "ouvidoria@adrianaealinecomerciodebebidasltda.com.br",
  "fiscal@nathaneleviadvocaciame.com.br",
  "producao@ritaeandreconsultoriafinanceirame.com.br",
  "seguranca@arthurealiciaconstrucoesltda.com.br",
  "marketing@joanaeeduardojoalheriame.com.br",
  "juridico@luiserobertoeletronicame.com.br",
  "fabricacao@emilyelorenatelecomme.com.br",
  "contato@martinebernardolimpezame.com.br",
  "seguranca@heloiseejaquelinemarketingme.com.br",
  "diretoria@gaelethomaspizzarialtda.com.br",
  "diretoria@adrianaelauratelecomunicacoesme.com.br",
  "treinamento@heloisaenicolemarketingme.com.br",
  "tesouraria@sueliepedrojoalheriame.com.br",
  "suporte@marcioebeneditoalimentosltda.com.br",
  "representantes@pietraenathanfilmagensme.com.br",
  "fabricacao@marliesergiofotografiasltda.com.br",
  "posvenda@isisebryanjoalheriame.com.br",
  "ti@gaelemanuelconsultoriafinanceirame.com.br",
  "fabricacao@brendaeemanuellyalimentosltda.com.br",
  "producao@renataepietroesportesme.com.br",
  "contabilidade@milenaesebastiaocomerciodebebidasltda.com.br",
  "almoxarifado@lizereginacomerciodebebidasltda.com.br",
  "fiscal@muriloefabioinformaticame.com.br",
  "almoxarifado@tomaseviniciusfotografiasme.com.br",
  "qualidade@brenoemurilopublicidadeepropagandame.com.br",
  "presidencia@luciaeanalueletronicaltda.com.br",
  "seguranca@nelsonemarcoslimpezame.com.br",
  "financeiro@henryegustavoferragensltda.com.br",
  "financeiro@martinebetinavidrosme.com.br",
  "marketing@samuelepietrocomerciodebebidasme.com.br",
  "orcamento@igoremanuelmudancasme.com.br",
  "contato@giovannievalentinacasanoturnaltda.com.br",
  "fiscal@patriciaeeduardadocessalgadosme.com.br",
  "vendas@giovannieaylaconsultoriafinanceiraltda.com.br",
  "desenvolvimento@calebeeyasmincasanoturname.com.br",
  "qualidade@isabelaeeliastelasltda.com.br",
  "financeiro@liviaebenicioferragensltda.com.br",
  "comunicacoes@ryaneeliastelecomltda.com.br",
  "rh@fernandoegabrielpadarialtda.com.br",
  "administracao@lucianaesebastiaoadvocaciame.com.br",
  "producao@hugoemariahgraficaltda.com.br",
];
function generateEmails() {
  return emails[getRandomInt(0, emails.length - 1)];
}

const phones = [
  "11995333483",
  "18982026305",
  "19989371250",
  "11993860173",
  "11984644466",
  "19992704578",
  "14984523236",
  "11986217867",
  "17987005166",
  "16981918829",
  "14981724025",
  "11998123161",
  "18996908666",
  "11984720429",
  "17993797698",
  "11985216521",
  "17987404736",
  "19991679604",
  "11999214750",
  "19981601954",
  "19981166928",
  "14991795701",
  "11992511797",
  "15995426300",
  "19981417684",
  "11989605586",
  "19993902987",
  "11984146333",
  "11996764895",
  "11997722499",
  "19984213512",
  "19992383179",
  "16989706731",
  "19984348172",
  "14985108178",
  "19989969421",
  "11999255505",
  "19995576735",
  "11989562262",
  "12986674514",
  "15989187962",
  "11987018094",
  "11995011535",
  "11987695596",
  "13984889196",
  "19985316190",
  "11985268568",
  "11983721253",
  "11997230208",
  "11981001798",
  "11996409607",
  "15996126142",
  "12988958769",
  "11984810714",
  "19998427959",
  "11981375692",
  "11996141703",
  "17986146259",
  "11985886824",
  "19986577778",
  "11996585490",
  "11995655467",
  "14996116494",
  "19997786094",
  "11986519122",
  "11997152858",
  "15981120391",
  "14993700507",
  "19985561176",
  "18992577783",
  "19999632099",
  "11983008592",
  "11994757416",
  "16994296901",
  "11988003951",
  "11998016244",
  "11991457390",
  "19989843078",
  "13984165272",
  "11983622241",
  "11997741801",
  "19998998101",
  "17988092469",
  "19992176730",
  "14981476389",
  "11991085159",
  "16997962084",
  "16998387632",
  "19983818985",
  "16982642241",
  "11982189329",
  "14983250851",
  "11983871018",
  "11991841520",
  "13983586651",
  "11987370094",
  "11985357537",
  "11982091404",
  "19998544072",
  "16982160641",
  "15986546798",
  "11984820065",
  "11992695922",
  "11985819414",
  "11994323016",
  "11997375737",
  "19982331714",
  "16992077319",
  "11985599037",
  "16982452463",
  "12986421573",
  "14993448614",
  "18987582313",
  "19982815323",
  "19991207180",
  "11981314451",
  "11989636762",
  "16992839900",
  "19994419540",
  "19982541016",
  "11994797737",
  "19991917692",
  "11982449213",
  "11988445430",
  "11989541300",
  "11985653206",
  "11986279458",
  "19999043930",
  "19983496963",
  "17994845161",
  "11994937064",
  "18993980432",
  "11992508613",
  "12986781577",
  "11982055854",
  "17983785641",
  "17993011845",
  "11995557150",
  "12987360479",
  "14981067730",
  "11989613834",
  "11993877451",
  "11994547796",
  "11988845168",
  "14989969689",
  "17999480433",
  "11999348657",
  "11992724540",
  "18998745642",
  "11985639895",
  "11997215898",
  "11986313408",
  "17999109033",
  "16993723495",
  "11984728191",
];
function generatePhone() {
  return phones[getRandomInt(0, phones.length - 1)];
}

const rgs = [
  "284268914",
  "358964003",
  "226272928",
  "428751921",
  "335107114",
  "464206169",
  "474293178",
  "232965651",
  "460372373",
  "340886730",
  "229558343",
  "499354771",
  "105563419",
  "302252745",
  "392278467",
  "286451815",
  "425298693",
  "195249409",
  "290528513",
  "441114179",
  "329462805",
  "206138647",
  "314177279",
  "390191310",
  "307373344",
  "305525232",
  "155172815",
  "451841517",
  "439935763",
  "416506641",
  "340337588",
  "130484118",
  "350363821",
  "491602704",
  "368052102",
  "416699972",
  "361725061",
  "394085231",
  "371023944",
  "144599946",
  "155560050",
  "189727317",
  "144141498",
  "194022146",
  "476497954",
  "419594012",
  "371418513",
  "456713025",
  "357031878",
  "333243407",
  "500861250",
  "485679991",
  "138737708",
  "465322980",
  "416219512",
  "200290459",
  "119804153",
  "226456985",
  "118463196",
  "157654035",
  "169752392",
  "205766055",
  "121965041",
  "179878864",
  "297173637",
  "103298198",
  "278541124",
  "501057614",
  "247328133",
  "205370081",
  "198888545",
  "429145172",
  "227240467",
  "300855278",
  "117218558",
  "487892987",
  "461997101",
  "478466328",
  "228796969",
  "279999057",
  "272449258",
  "238833495",
  "502323723",
  "498213511",
  "293682318",
  "441231548",
  "232406972",
  "388441033",
  "363166348",
  "290453641",
  "279669781",
  "282688523",
  "326700997",
  "384235049",
  "356734638",
  "193210447",
  "158048854",
  "101293173",
  "344624067",
  "366693013",
  "125702656",
  "161808001",
  "475521912",
  "138764207",
  "389914423",
  "385051207",
  "382609694",
  "162358520",
  "167205195",
  "274840819",
  "263340065",
  "246427760",
  "286420983",
  "433894404",
  "297003811",
  "132103217",
  "349545212",
  "310075841",
  "215756022",
  "501140736",
  "181812472",
  "453715564",
  "305330494",
  "324372814",
  "347169399",
  "306560501",
  "185884581",
  "146160198",
  "147788985",
  "304784874",
  "303205593",
  "402587005",
  "338147214",
  "467177612",
  "361331757",
  "366756424",
  "221342308",
  "277873484",
  "138916998",
  "465868381",
  "435292973",
  "101665398",
  "359876018",
  "181697671",
  "493808590",
  "459561182",
  "242216894",
  "206859235",
  "122595737",
  "402658024",
  "360481152",
  "175308706",
  "295458690",
  "275294717",
  "471448904",
  "418230523",
  "167609117",
  "326077984",
  "187245423",
  "439905126",
  "402285025",
  "496091220",
  "223305935",
  "114602190",
  "186622934",
  "405541983",
  "503320729",
  "235692177",
  "371633680",
  "289914784",
  "462728018",
  "200728866",
  "160388259",
  "288861395",
  "458066072",
  "343166896",
  "336410852",
  "124318782",
  "113815682",
  "488064211",
  "349408348",
  "389115940",
  "174117218",
  "469522082",
  "480742662",
  "374024297",
  "232369537",
  "272272796",
  "333749844",
  "246272636",
  "390222513",
  "447360280",
  "245611903",
  "241083175",
  "408482618",
  "436401034",
  "182350708",
  "234265929",
  "352301624",
  "267630657",
  "480821458",
  "170853913",
  "176232564",
  "358357676",
  "353690260",
  "457686091",
  "446299571",
  "274874593",
  "328128223",
  "229893880",
  "102991479",
  "194109367",
  "433420601",
  "125300621",
  "222603811",
  "446104553",
  "236817735",
  "407014159",
  "303379078",
  "346611507",
  "350859061",
  "291082798",
  "481512688",
  "462579438",
  "376613154",
  "445583472",
  "477435762",
  "324066272",
  "221303728",
  "121387963",
  "137781416",
  "300709699",
  "314865603",
  "149806838",
  "117309692",
  "351577300",
  "332261748",
  "101297889",
  "400606069",
  "354344407",
  "463909434",
  "132665785",
  "133559294",
  "485064820",
  "373078985",
  "201891268",
  "225441743",
  "216693883",
  "235306277",
  "312610993",
  "119837572",
  "308811380",
  "109125885",
  "239861152",
  "124262351",
  "358653460",
  "504792519",
  "394065682",
  "247490155",
  "163163984",
  "158400033",
  "198060774",
  "341724828",
  "122528864",
  "301348984",
  "205289137",
  "245762395",
  "338055411",
  "228392287",
  "502194625",
  "207979911",
  "214657413",
  "500412650",
  "369931038",
  "339391601",
  "237569954",
  "347284991",
  "417643676",
  "500560997",
  "473311069",
  "148273440",
  "168187759",
  "405338703",
  "287218338",
  "293608969",
  "367705199",
  "132414764",
  "354897998",
  "243803187",
  "303659439",
  "278915474",
  "386012702",
  "219059299",
  "263575317",
  "491300281",
  "330356793",
  "258391297",
  "107974526",
  "116589644",
  "397847488",
  "350210871",
  "487355441",
  "255842557",
  "424587877",
  "374793566",
  "244439448",
  "312646896",
  "451488532",
  "259298402",
  "137688271",
  "182415296",
  "289176323",
  "258607671",
  "349924867",
  "450381080",
  "123206741",
  "348373740",
  "177941224",
  "305377358",
  "472448973",
  "199634762",
  "105688289",
  "509157968",
  "449152480",
  "179932949",
  "378797232",
  "202456122",
  "208859330",
  "118678292",
  "389518803",
  "451405602",
  "122790571",
  "436720103",
  "497530065",
  "432252575",
  "229886474",
  "107707974",
  "363573069",
  "123518106",
  "173621259",
  "267658606",
  "204256690",
  "488010809",
  "168807683",
  "260893936",
  "387740314",
  "335391679",
  "306097059",
  "364806667",
  "367618357",
  "484482300",
  "252170787",
  "136293037",
  "478779288",
  "405639338",
  "428464907",
  "477702508",
  "395742778",
  "154078645",
  "223313890",
  "507762848",
  "126662988",
  "112596939",
  "364747365",
  "221696623",
  "205508224",
  "460328268",
  "177543747",
  "288482025",
  "426123591",
  "111651293",
  "465042399",
  "207359933",
  "148748120",
  "444329225",
  "378528464",
  "310387905",
  "436072981",
  "252454741",
  "148649191",
  "426146694",
  "414917510",
  "480662903",
  "290286505",
  "313589860",
  "500813486",
  "253990130",
  "249641264",
  "252519292",
  "370200202",
  "392136247",
  "354193016",
  "377543421",
  "190174286",
  "189955764",
  "371451899",
  "409668199",
  "330028819",
  "458617714",
  "397110686",
  "502002670",
  "130721797",
  "259392881",
  "464120408",
  "114433446",
  "135424653",
  "369832140",
  "455096831",
  "452758713",
  "321755029",
  "481681681",
  "297085050",
  "474959223",
  "304860049",
  "500493728",
  "452465345",
  "292666998",
  "210835266",
  "414472512",
  "297847211",
  "157258890",
  "398242586",
  "201093388",
  "426071256",
  "410667043",
  "194432026",
  "483788338",
  "183389670",
  "242727670",
  "212156627",
  "131092807",
  "105140521",
  "425304978",
  "110220766",
  "174992270",
  "408296720",
  "217721527",
  "119246661",
  "269156136",
  "404095331",
  "427786897",
  "124956178",
  "409838354",
  "337752096",
  "363044875",
  "492106617",
  "311104058",
  "454490884",
  "230821704",
  "445182192",
  "406453962",
  "453457411",
  "130492218",
  "412629677",
  "362319303",
  "169918993",
  "182971843",
  "293951329",
  "239581027",
  "385616995",
  "127182780",
  "243706509",
  "160188209",
  "418654335",
  "459664359",
  "295053318",
  "261597358",
  "227891454",
  "446918210",
  "310977599",
  "103716828",
  "184940539",
  "128367258",
  "478099666",
  "318287572",
  "251891835",
  "219332940",
  "145684039",
  "168280656",
  "207821562",
  "155613200",
  "171680455",
  "193671281",
  "492817571",
  "238538345",
  "333867737",
  "424050195",
  "377343468",
  "235251458",
  "362877105",
  "275698890",
  "494635812",
  "156224483",
  "131803700",
  "284330243",
  "395262483",
  "156256885",
  "137600124",
  "370595403",
  "124862287",
];
function generateRG() {
  return rgs[getRandomInt(0, rgs.length - 1)];
}
const enterprise = [
  {
    name: "Rafael e Raul Financeira ME",
    cnpj: "60665636000184",
    registerState: "442826596571",
    dateOpen: "25/03/2018",
    site: "www.rafaeleraulfinanceirame.com.br",
    email: "seguranca@rafaeleraulfinanceirame.com.br",
    cep: "04011900",
    street: "Rua José Antônio Coelho 300",
    number: "765",
    neighborhood: "Vila Mariana",
    city: "São Paulo",
    state: "SP",
    phone: "11995333483",
  },
  {
    name: "Sophie e Cláudia Corretores Associados Ltda",
    cnpj: "83154448000197",
    registerState: "664033387015",
    dateOpen: "16/04/2018",
    site: "www.sophieeclaudiacorretoresassociadosltda.com.br",
    email: "desenvolvimento@sophieeclaudiacorretoresassociadosltda.com.br",
    cep: "16200123",
    street: "Avenida Afonso Pena",
    number: "946",
    neighborhood: "Vila Pontes",
    city: "Birigüi",
    state: "SP",
    phone: "18982026305",
  },
  {
    name: "Beatriz e Leonardo Entregas Expressas ME",
    cnpj: "92777232000179",
    registerState: "399768975525",
    dateOpen: "10/08/2018",
    site: "www.beatrizeleonardoentregasexpressasme.com.br",
    email: "faleconosco@beatrizeleonardoentregasexpressasme.com.br",
    cep: "13635038",
    street: "Rua Lydia de Arruda Reis",
    number: "221",
    neighborhood: "Jardim São Valentim",
    city: "Pirassununga",
    state: "SP",
    phone: "19989371250",
  },
  {
    name: "Sueli e Luan Pizzaria ME",
    cnpj: "78167457000182",
    registerState: "019846536588",
    dateOpen: "02/01/2018",
    site: "www.suelieluanpizzariame.com.br",
    email: "diretoria@suelieluanpizzariame.com.br",
    cep: "06775260",
    street: "Rua Alcides Luizetto",
    number: "912",
    neighborhood: "Jardim Guaciara",
    city: "Taboão da Serra",
    state: "SP",
    phone: "11993860173",
  },
  {
    name: "Elisa e Geraldo Informática ME",
    cnpj: "21894256000160",
    registerState: "605158140359",
    dateOpen: "10/08/2018",
    site: "www.elisaegeraldoinformaticame.com.br",
    email: "tesouraria@elisaegeraldoinformaticame.com.br",
    cep: "12951086",
    street: "Rua José Abramo",
    number: "163",
    neighborhood: "Casas Populares",
    city: "Atibaia",
    state: "SP",
    phone: "11984644466",
  },
  {
    name: "Iago e Manuel Limpeza Ltda",
    cnpj: "65345011000159",
    registerState: "873628832062",
    dateOpen: "10/02/2018",
    site: "www.iagoemanuellimpezaltda.com.br",
    email: "treinamento@iagoemanuellimpezaltda.com.br",
    cep: "13489002",
    street: "Rua Emílio Pantano",
    number: "378",
    neighborhood: "Tatu",
    city: "Limeira",
    state: "SP",
    phone: "19992704578",
  },
  {
    name: "Cláudio e Benjamin Consultoria Financeira Ltda",
    cnpj: "05159272000152",
    registerState: "402991995052",
    dateOpen: "25/03/2018",
    site: "www.claudioebenjaminconsultoriafinanceiraltda.com.br",
    email: "contato@claudioebenjaminconsultoriafinanceiraltda.com.br",
    cep: "16400793",
    street: "Rua Santa Terezinha",
    number: "495",
    neighborhood: "Vila Pacini",
    city: "Lins",
    state: "SP",
    phone: "14984523236",
  },
  {
    name: "Giovana e João Lavanderia Ltda",
    cnpj: "29985902000106",
    registerState: "818763156287",
    dateOpen: "05/10/2018",
    site: "www.giovanaejoaolavanderialtda.com.br",
    email: "auditoria@giovanaejoaolavanderialtda.com.br",
    cep: "02965110",
    street: "Rua Francisco de Paula Moura Neto",
    number: "888",
    neighborhood: "Vila Iório",
    city: "São Paulo",
    state: "SP",
    phone: "11986217867",
  },
  {
    name: "João e Cauê Pizzaria Ltda",
    cnpj: "42084760000174",
    registerState: "896962989396",
    dateOpen: "07/11/2018",
    site: "www.joaoecauepizzarialtda.com.br",
    email: "representantes@joaoecauepizzarialtda.com.br",
    cep: "15030790",
    street: "Avenida Philadelpho Manoel Gouveia Netto",
    number: "477",
    neighborhood: "Jardim Conceição",
    city: "São José do Rio Preto",
    state: "SP",
    phone: "17987005166",
  },
  {
    name: "Nina e Cauã Telas Ltda",
    cnpj: "14793423000110",
    registerState: "289114777022",
    dateOpen: "17/10/2018",
    site: "www.ninaecauatelasltda.com.br",
    email: "fabricacao@ninaecauatelasltda.com.br",
    cep: "14025415",
    street: "Rua Vicente de Carvalho",
    number: "533",
    neighborhood: "Jardim Sumaré",
    city: "Ribeirão Preto",
    state: "SP",
    phone: "16981918829",
  },
  {
    name: "Gabriela e Amanda Esportes ME",
    cnpj: "96058732000101",
    registerState: "695081151431",
    dateOpen: "05/12/2018",
    site: "www.gabrielaeamandaesportesme.com.br",
    email: "auditoria@gabrielaeamandaesportesme.com.br",
    cep: "16520970",
    street: "Rua 12 de Outubro, s/n",
    number: "240",
    neighborhood: "Centro",
    city: "Bacuriti",
    state: "SP",
    phone: "14981724025",
  },
  {
    name: "Vanessa e Alícia Pizzaria ME",
    cnpj: "68967036000182",
    registerState: "890278491118",
    dateOpen: "10/07/2018",
    site: "www.vanessaealiciapizzariame.com.br",
    email: "suporte@vanessaealiciapizzariame.com.br",
    cep: "02680080",
    street: "Avenida Arquiteto Roberto Aflalo",
    number: "350",
    neighborhood: "Jardim Peri",
    city: "São Paulo",
    state: "SP",
    phone: "11998123161",
  },
  {
    name: "Renan e Camila Alimentos ME",
    cnpj: "92107873000116",
    registerState: "697019106680",
    dateOpen: "13/06/2018",
    site: "www.renanecamilaalimentosme.com.br",
    email: "treinamento@renanecamilaalimentosme.com.br",
    cep: "16078010",
    street: "Rua Pedro Storti",
    number: "164",
    neighborhood: "Parque Industrial",
    city: "Araçatuba",
    state: "SP",
    phone: "18996908666",
  },
  {
    name: "Gustavo e Luzia Consultoria Financeira Ltda",
    cnpj: "29953436000187",
    registerState: "777988753875",
    dateOpen: "21/02/2018",
    site: "www.gustavoeluziaconsultoriafinanceiraltda.com.br",
    email: "contabil@gustavoeluziaconsultoriafinanceiraltda.com.br",
    cep: "07182130",
    street: "Rua Bento Gonçalves",
    number: "512",
    neighborhood: "Cidade SOIMCO",
    city: "Guarulhos",
    state: "SP",
    phone: "11984720429",
  },
  {
    name: "Stefany e Elias Corretores Associados ME",
    cnpj: "32553492000139",
    registerState: "148249418323",
    dateOpen: "09/03/2018",
    site: "www.stefanyeeliascorretoresassociadosme.com.br",
    email: "contato@stefanyeeliascorretoresassociadosme.com.br",
    cep: "15051600",
    street: "Rua Projetada A",
    number: "342",
    neighborhood: "Mariza Cristina I",
    city: "São José do Rio Preto",
    state: "SP",
    phone: "17993797698",
  },
  {
    name: "Luiza e Enzo Mudanças Ltda",
    cnpj: "86718909000103",
    registerState: "762238503640",
    dateOpen: "03/08/2018",
    site: "www.luizaeenzomudancasltda.com.br",
    email: "faleconosco@luizaeenzomudancasltda.com.br",
    cep: "02834020",
    street: "Rua José de Campos Novais",
    number: "388",
    neighborhood: "Vila Nina",
    city: "São Paulo",
    state: "SP",
    phone: "11985216521",
  },
  {
    name: "Allana e Thomas Telecomunicações Ltda",
    cnpj: "46109831000105",
    registerState: "316731703151",
    dateOpen: "10/06/2018",
    site: "www.allanaethomastelecomunicacoesltda.com.br",
    email: "ti@allanaethomastelecomunicacoesltda.com.br",
    cep: "15042065",
    street: "Rua Adelino Primo Malagoli",
    number: "212",
    neighborhood: "Residencial Gabriela",
    city: "São José do Rio Preto",
    state: "SP",
    phone: "17987404736",
  },
  {
    name: "Jennifer e André Marcenaria ME",
    cnpj: "67932066000190",
    registerState: "340362420604",
    dateOpen: "15/05/2018",
    site: "www.jennifereandremarcenariame.com.br",
    email: "atendimento@jennifereandremarcenariame.com.br",
    cep: "13049374",
    street: "Rua Felipe Ricci",
    number: "306",
    neighborhood: "Swiss Park",
    city: "Campinas",
    state: "SP",
    phone: "19991679604",
  },
  {
    name: "Agatha e Emily Entregas Expressas Ltda",
    cnpj: "54332657000185",
    registerState: "637640462577",
    dateOpen: "16/05/2018",
    site: "www.agathaeemilyentregasexpressasltda.com.br",
    email: "administracao@agathaeemilyentregasexpressasltda.com.br",
    cep: "07914090",
    street: "Rua Vantuin Aparecido Martins",
    number: "895",
    neighborhood: "Jardim Vagliengo",
    city: "Francisco Morato",
    state: "SP",
    phone: "11999214750",
  },
  {
    name: "Rita e Isaac Telas Ltda",
    cnpj: "27467016000165",
    registerState: "158991180579",
    dateOpen: "06/02/2018",
    site: "www.ritaeisaactelasltda.com.br",
    email: "rh@ritaeisaactelasltda.com.br",
    cep: "13455715",
    street: "Rua Oswaldo Lacava",
    number: "221",
    neighborhood: "Jardim Dona Regina",
    city: "Santa Bárbara D'Oeste",
    state: "SP",
    phone: "19981601954",
  },
  {
    name: "Louise e Enrico Telecom Ltda",
    cnpj: "92592075000127",
    registerState: "177940978336",
    dateOpen: "01/08/2018",
    site: "www.louiseeenricotelecomltda.com.br",
    email: "suporte@louiseeenricotelecomltda.com.br",
    cep: "13087582",
    street: "Rua Sargento Bombeiro Assiz Degrossoli Filho",
    number: "541",
    neighborhood: "Parque Rural Fazenda Santa Cândida",
    city: "Campinas",
    state: "SP",
    phone: "19981166928",
  },
  {
    name: "Emanuel e Ruan Pães e Doces ME",
    cnpj: "41144317000189",
    registerState: "956864221755",
    dateOpen: "25/11/2018",
    site: "www.emanueleruanpaesedocesme.com.br",
    email: "treinamento@emanueleruanpaesedocesme.com.br",
    cep: "17210700",
    street: "Rua João Ronchesel",
    number: "163",
    neighborhood: "Jardim América",
    city: "Jaú",
    state: "SP",
    phone: "14991795701",
  },
  {
    name: "Sérgio e Helena Ferragens ME",
    cnpj: "75275032000107",
    registerState: "713178389764",
    dateOpen: "27/01/2018",
    site: "www.sergioehelenaferragensme.com.br",
    email: "qualidade@sergioehelenaferragensme.com.br",
    cep: "04212001",
    street: "Rua Almirante Lobo",
    number: "887",
    neighborhood: "Ipiranga",
    city: "São Paulo",
    state: "SP",
    phone: "11992511797",
  },
  {
    name: "Igor e Isis Telas ME",
    cnpj: "32698329000164",
    registerState: "161812130999",
    dateOpen: "20/02/2018",
    site: "www.igoreisistelasme.com.br",
    email: "estoque@igoreisistelasme.com.br",
    cep: "18103440",
    street: "Rua Ascencion Desojo Dias",
    number: "524",
    neighborhood: "Jardim Jatobá",
    city: "Sorocaba",
    state: "SP",
    phone: "15995426300",
  },
  {
    name: "Luana e Carla Locações de Automóveis ME",
    cnpj: "27935254000158",
    registerState: "250269770622",
    dateOpen: "09/01/2018",
    site: "www.luanaecarlalocacoesdeautomoveisme.com.br",
    email: "seguranca@luanaecarlalocacoesdeautomoveisme.com.br",
    cep: "13482173",
    street: "Rua Maria Demo Cason",
    number: "744",
    neighborhood: "Jardim Adélia Cavicchia Grotta",
    city: "Limeira",
    state: "SP",
    phone: "19981417684",
  },
  {
    name: "Caio e Lucca Pães e Doces Ltda",
    cnpj: "77853490000101",
    registerState: "690567297473",
    dateOpen: "21/12/2018",
    site: "www.caioeluccapaesedocesltda.com.br",
    email: "producao@caioeluccapaesedocesltda.com.br",
    cep: "09669100",
    street: "Avenida Água Funda",
    number: "540",
    neighborhood: "Taboão",
    city: "São Bernardo do Campo",
    state: "SP",
    phone: "11989605586",
  },
  {
    name: "Leonardo e Pedro Henrique Contábil ME",
    cnpj: "16350268000193",
    registerState: "991036483230",
    dateOpen: "23/02/2018",
    site: "www.leonardoepedrohenriquecontabilme.com.br",
    email: "financeiro@leonardoepedrohenriquecontabilme.com.br",
    cep: "13466030",
    street: "Rua Cabo Oswaldo de Moraes",
    number: "707",
    neighborhood: "Conserva",
    city: "Americana",
    state: "SP",
    phone: "19993902987",
  },
  {
    name: "Agatha e Leandro Buffet ME",
    cnpj: "30637522000160",
    registerState: "276599030371",
    dateOpen: "23/02/2018",
    site: "www.agathaeleandrobuffetme.com.br",
    email: "atendimento@agathaeleandrobuffetme.com.br",
    cep: "07033090",
    street: "Rua Doutor João Ayres de Queiroz",
    number: "443",
    neighborhood: "Vila Clotilde",
    city: "Guarulhos",
    state: "SP",
    phone: "11984146333",
  },
  {
    name: "Fátima e Tânia Publicidade e Propaganda ME",
    cnpj: "68147562000104",
    registerState: "551748363326",
    dateOpen: "25/03/2018",
    site: "www.fatimaetaniapublicidadeepropagandame.com.br",
    email: "almoxarifado@fatimaetaniapublicidadeepropagandame.com.br",
    cep: "09240150",
    street: "Rua Teresópolis",
    number: "828",
    neighborhood: "Jardim Santo Antônio",
    city: "Santo André",
    state: "SP",
    phone: "11996764895",
  },
  {
    name: "Luan e Priscila Padaria ME",
    cnpj: "44969678000152",
    registerState: "577395928746",
    dateOpen: "02/06/2018",
    site: "www.luanepriscilapadariame.com.br",
    email: "representantes@luanepriscilapadariame.com.br",
    cep: "09445230",
    street: "Rua Jupiá",
    number: "560",
    neighborhood: "Somma",
    city: "Ribeirão Pires",
    state: "SP",
    phone: "11997722499",
  },
  {
    name: "Bento e Gabriel Filmagens ME",
    cnpj: "82546283000136",
    registerState: "074394759277",
    dateOpen: "05/09/2018",
    site: "www.bentoegabrielfilmagensme.com.br",
    email: "ti@bentoegabrielfilmagensme.com.br",
    cep: "13091207",
    street: "Rua Jupira Paulina da Silva",
    number: "484",
    neighborhood: "Jardim Novo Flamboyant",
    city: "Campinas",
    state: "SP",
    phone: "19984213512",
  },
  {
    name: "Renan e Daniela Casa Noturna ME",
    cnpj: "30134965000138",
    registerState: "506043044400",
    dateOpen: "27/10/2018",
    site: "www.renanedanielacasanoturname.com.br",
    email: "seguranca@renanedanielacasanoturname.com.br",
    cep: "13474780",
    street: "Avenida do Algodão",
    number: "480",
    neighborhood: "Loteamento Industrial Salto Grande I",
    city: "Americana",
    state: "SP",
    phone: "19992383179",
  },
  {
    name: "Sara e Levi Restaurante Ltda",
    cnpj: "23026888000137",
    registerState: "611051822553",
    dateOpen: "10/09/2018",
    site: "www.saraelevirestauranteltda.com.br",
    email: "producao@saraelevirestauranteltda.com.br",
    cep: "14804254",
    street: "Avenida Geraldo Neves Júnior",
    number: "181",
    neighborhood: "Residencial Acapulco",
    city: "Araraquara",
    state: "SP",
    phone: "16989706731",
  },
  {
    name: "Vitor e Joana Construções ME",
    cnpj: "47476036000100",
    registerState: "595550132400",
    dateOpen: "15/08/2018",
    site: "www.vitorejoanaconstrucoesme.com.br",
    email: "seguranca@vitorejoanaconstrucoesme.com.br",
    cep: "13607150",
    street: "Rua Paschoal Battel",
    number: "331",
    neighborhood: "Jardim Santa Marta",
    city: "Araras",
    state: "SP",
    phone: "19984348172",
  },
  {
    name: "Gabriela e Marina Consultoria Financeira Ltda",
    cnpj: "36538477000172",
    registerState: "794292911764",
    dateOpen: "08/11/2018",
    site: "www.gabrielaemarinaconsultoriafinanceiraltda.com.br",
    email: "ouvidoria@gabrielaemarinaconsultoriafinanceiraltda.com.br",
    cep: "17050566",
    street: "Rua Coronel Gomes dos Santos",
    number: "428",
    neighborhood: "Vila São Manoel",
    city: "Bauru",
    state: "SP",
    phone: "14985108178",
  },
  {
    name: "Marcela e Henrique Vidros Ltda",
    cnpj: "09056356000111",
    registerState: "960959288035",
    dateOpen: "09/05/2018",
    site: "www.marcelaehenriquevidrosltda.com.br",
    email: "representantes@marcelaehenriquevidrosltda.com.br",
    cep: "13273091",
    street: "Rua Santo Tordin",
    number: "845",
    neighborhood: "Jardim dos Manacás",
    city: "Valinhos",
    state: "SP",
    phone: "19989969421",
  },
  {
    name: "Débora e Luciana Doces & Salgados ME",
    cnpj: "93731035000181",
    registerState: "814305534774",
    dateOpen: "26/04/2018",
    site: "www.deboraelucianadocessalgadosme.com.br",
    email: "auditoria@deboraelucianadocessalgadosme.com.br",
    cep: "03373125",
    street: "Rua Joaquim da Silva",
    number: "251",
    neighborhood: "Chácara Mafalda",
    city: "São Paulo",
    state: "SP",
    phone: "11999255505",
  },
  {
    name: "Raul e Manuela Comercio de Bebidas ME",
    cnpj: "39532873000180",
    registerState: "131210718593",
    dateOpen: "23/09/2018",
    site: "www.raulemanuelacomerciodebebidasme.com.br",
    email: "representantes@raulemanuelacomerciodebebidasme.com.br",
    cep: "13185350",
    street: "Rua Andrea da Silva Barros",
    number: "865",
    neighborhood: "Jardim Adelaide",
    city: "Hortolândia",
    state: "SP",
    phone: "19995576735",
  },
  {
    name: "Mário e Filipe Pizzaria Delivery Ltda",
    cnpj: "74201280000132",
    registerState: "308467727480",
    dateOpen: "02/06/2018",
    site: "www.marioefilipepizzariadeliveryltda.com.br",
    email: "ouvidoria@marioefilipepizzariadeliveryltda.com.br",
    cep: "08536520",
    street: "Rua Vereador Alexandre Secundino Leite",
    number: "763",
    neighborhood: "Jardim Angelina",
    city: "Ferraz de Vasconcelos",
    state: "SP",
    phone: "11989562262",
  },
  {
    name: "Vera e Fábio Pães e Doces ME",
    cnpj: "57013069000177",
    registerState: "456526590068",
    dateOpen: "16/05/2018",
    site: "www.veraefabiopaesedocesme.com.br",
    email: "producao@veraefabiopaesedocesme.com.br",
    cep: "11667730",
    street: "Avenida Casemiro José Marques de Abreu",
    number: "542",
    neighborhood: "Jardim Brasil",
    city: "Caraguatatuba",
    state: "SP",
    phone: "12986674514",
  },
  {
    name: "Julia e Carlos Doces & Salgados ME",
    cnpj: "97076325000190",
    registerState: "474410702729",
    dateOpen: "25/07/2018",
    site: "www.juliaecarlosdocessalgadosme.com.br",
    email: "ouvidoria@juliaecarlosdocessalgadosme.com.br",
    cep: "18304770",
    street: "Rua das Bromélias",
    number: "698",
    neighborhood: "Querência do Turvo",
    city: "Capão Bonito",
    state: "SP",
    phone: "15989187962",
  },
  {
    name: "Sophie e Lúcia Doces & Salgados Ltda",
    cnpj: "64166902000185",
    registerState: "430987109578",
    dateOpen: "27/11/2018",
    site: "www.sophieeluciadocessalgadosltda.com.br",
    email: "juridico@sophieeluciadocessalgadosltda.com.br",
    cep: "08690560",
    street: "Rua Cento e Seis",
    number: "779",
    neighborhood: "Cidade Miguel Badra",
    city: "Suzano",
    state: "SP",
    phone: "11987018094",
  },
  {
    name: "José e Manoel Construções ME",
    cnpj: "61485792000126",
    registerState: "967008340315",
    dateOpen: "25/03/2018",
    site: "www.joseemanoelconstrucoesme.com.br",
    email: "almoxarifado@joseemanoelconstrucoesme.com.br",
    cep: "06785360",
    street: "Rua Europa",
    number: "739",
    neighborhood: "Parque Industrial Daci",
    city: "Taboão da Serra",
    state: "SP",
    phone: "11995011535",
  },
  {
    name: "Isis e Thiago Lavanderia ME",
    cnpj: "49448042000189",
    registerState: "680858670241",
    dateOpen: "02/12/2018",
    site: "www.isisethiagolavanderiame.com.br",
    email: "fabricacao@isisethiagolavanderiame.com.br",
    cep: "08696555",
    street: "Rua Rômulo Giannichi",
    number: "763",
    neighborhood: "Jardim Chácara Méa",
    city: "Suzano",
    state: "SP",
    phone: "11987695596",
  },
  {
    name: "Mariane e Eliane Transportes ME",
    cnpj: "74202558000196",
    registerState: "666403251013",
    dateOpen: "08/12/2018",
    site: "www.marianeeelianetransportesme.com.br",
    email: "ouvidoria@marianeeelianetransportesme.com.br",
    cep: "11437585",
    street: "Viela do Reservatório I",
    number: "773",
    neighborhood: "Morro do Engenho",
    city: "Guarujá",
    state: "SP",
    phone: "13984889196",
  },
  {
    name: "Antônia e Hadassa Gráfica ME",
    cnpj: "34316188000101",
    registerState: "828382892936",
    dateOpen: "12/08/2018",
    site: "www.antoniaehadassagraficame.com.br",
    email: "producao@antoniaehadassagraficame.com.br",
    cep: "13063050",
    street: "Rua Altemiro de Sousa Leite",
    number: "196",
    neighborhood: "Jardim Eulina",
    city: "Campinas",
    state: "SP",
    phone: "19985316190",
  },
  {
    name: "Patrícia e Renan Consultoria Financeira Ltda",
    cnpj: "73984722000100",
    registerState: "102922040322",
    dateOpen: "14/08/2018",
    site: "www.patriciaerenanconsultoriafinanceiraltda.com.br",
    email: "contato@patriciaerenanconsultoriafinanceiraltda.com.br",
    cep: "01049907",
    street: "Rua Formosa 405",
    number: "936",
    neighborhood: "Centro",
    city: "São Paulo",
    state: "SP",
    phone: "11985268568",
  },
  {
    name: "Nelson e Julio Entregas Expressas ME",
    cnpj: "66652712000101",
    registerState: "719443544806",
    dateOpen: "14/01/2018",
    site: "www.nelsonejulioentregasexpressasme.com.br",
    email: "producao@nelsonejulioentregasexpressasme.com.br",
    cep: "01441020",
    street: "Rua Doutor Antônio Carlos de Assumpção",
    number: "521",
    neighborhood: "Jardim América",
    city: "São Paulo",
    state: "SP",
    phone: "11983721253",
  },
  {
    name: "Luna e Enzo Locações de Automóveis ME",
    cnpj: "37543264000100",
    registerState: "945979403002",
    dateOpen: "26/09/2018",
    site: "www.lunaeenzolocacoesdeautomoveisme.com.br",
    email: "vendas@lunaeenzolocacoesdeautomoveisme.com.br",
    cep: "08573520",
    street: "Avenida Brasil",
    number: "379",
    neighborhood: "Jardim Gonçalves",
    city: "Itaquaquecetuba",
    state: "SP",
    phone: "11997230208",
  },
  {
    name: "Lucca e Noah Marcenaria ME",
    cnpj: "22275639000112",
    registerState: "648560118905",
    dateOpen: "10/12/2018",
    site: "www.luccaenoahmarcenariame.com.br",
    email: "representantes@luccaenoahmarcenariame.com.br",
    cep: "05429900",
    street: "Rua Costa Carvalho 300",
    number: "159",
    neighborhood: "Pinheiros",
    city: "São Paulo",
    state: "SP",
    phone: "11981001798",
  },
  {
    name: "Alana e Lúcia Joalheria ME",
    cnpj: "18360803000112",
    registerState: "264775413243",
    dateOpen: "21/08/2018",
    site: "www.alanaeluciajoalheriame.com.br",
    email: "auditoria@alanaeluciajoalheriame.com.br",
    cep: "13203272",
    street: "Rua João Batista de Morais",
    number: "358",
    neighborhood: "Conjunto Residencial Iapi",
    city: "Jundiaí",
    state: "SP",
    phone: "11996409607",
  },
  {
    name: "Teresinha e Sophia Telas Ltda",
    cnpj: "75180196000142",
    registerState: "298479490800",
    dateOpen: "03/06/2018",
    site: "www.teresinhaesophiatelasltda.com.br",
    email: "administracao@teresinhaesophiatelasltda.com.br",
    cep: "18204050",
    street: "Rua Água Marinha",
    number: "706",
    neighborhood: "Vale San Fernando",
    city: "Itapetininga",
    state: "SP",
    phone: "15996126142",
  },
  {
    name: "Malu e Caio Construções ME",
    cnpj: "72681549000108",
    registerState: "175614406421",
    dateOpen: "20/08/2018",
    site: "www.maluecaioconstrucoesme.com.br",
    email: "ouvidoria@maluecaioconstrucoesme.com.br",
    cep: "12040770",
    street: "Rua Reginaldo Antônio Sandy",
    number: "907",
    neighborhood: "Esplanada Independência",
    city: "Taubaté",
    state: "SP",
    phone: "12988958769",
  },
  {
    name: "Fernanda e Alícia Limpeza ME",
    cnpj: "45936052000102",
    registerState: "888539005355",
    dateOpen: "15/02/2018",
    site: "www.fernandaealicialimpezame.com.br",
    email: "manutencao@fernandaealicialimpezame.com.br",
    cep: "09941655",
    street: "Travessa Doutor Humberto",
    number: "723",
    neighborhood: "Canhema",
    city: "Diadema",
    state: "SP",
    phone: "11984810714",
  },
  {
    name: "Isabella e Iago Assessoria Jurídica ME",
    cnpj: "70761597000172",
    registerState: "748666200920",
    dateOpen: "25/01/2018",
    site: "www.isabellaeiagoassessoriajuridicame.com.br",
    email: "pesquisa@isabellaeiagoassessoriajuridicame.com.br",
    cep: "13800723",
    street: "Rua Hilário Bataglini",
    number: "271",
    neighborhood: "Saúde",
    city: "Mogi Mirim",
    state: "SP",
    phone: "19998427959",
  },
  {
    name: "Priscila e Andrea Doces & Salgados Ltda",
    cnpj: "71876455000113",
    registerState: "618108713934",
    dateOpen: "23/07/2018",
    site: "www.priscilaeandreadocessalgadosltda.com.br",
    email: "financeiro@priscilaeandreadocessalgadosltda.com.br",
    cep: "06663620",
    street: "Rua Marajó",
    number: "693",
    neighborhood: "Parque Suburbano",
    city: "Itapevi",
    state: "SP",
    phone: "11981375692",
  },
  {
    name: "Silvana e Vinicius Pizzaria Delivery ME",
    cnpj: "18501145000131",
    registerState: "064267977800",
    dateOpen: "26/09/2018",
    site: "www.silvanaeviniciuspizzariadeliveryme.com.br",
    email: "cobranca@silvanaeviniciuspizzariadeliveryme.com.br",
    cep: "06877150",
    street: "Rua Antônio de Albuquerque",
    number: "734",
    neighborhood: "Aldeinha",
    city: "Itapecerica da Serra",
    state: "SP",
    phone: "11996141703",
  },
  {
    name: "Maitê e Aurora Informática ME",
    cnpj: "85627643000112",
    registerState: "670719783330",
    dateOpen: "09/04/2018",
    site: "www.maiteeaurorainformaticame.com.br",
    email: "cobranca@maiteeaurorainformaticame.com.br",
    cep: "15047230",
    street: "Avenida D",
    number: "843",
    neighborhood: "Parque da Cidadania",
    city: "São José do Rio Preto",
    state: "SP",
    phone: "17986146259",
  },
  {
    name: "Henrique e Sophia Financeira Ltda",
    cnpj: "15276167000157",
    registerState: "017170590556",
    dateOpen: "22/11/2018",
    site: "www.henriqueesophiafinanceiraltda.com.br",
    email: "estoque@henriqueesophiafinanceiraltda.com.br",
    cep: "08395170",
    street: "Travessa Augusto Baron",
    number: "740",
    neighborhood: "Conjunto Promorar Rio Claro",
    city: "São Paulo",
    state: "SP",
    phone: "11985886824",
  },
  {
    name: "Eduarda e José Entulhos ME",
    cnpj: "33319712000127",
    registerState: "201392066410",
    dateOpen: "27/09/2018",
    site: "www.eduardaejoseentulhosme.com.br",
    email: "estoque@eduardaejoseentulhosme.com.br",
    cep: "13476643",
    street: "Rua João Ranieri",
    number: "487",
    neighborhood: "Parque Dom Pedro II",
    city: "Americana",
    state: "SP",
    phone: "19986577778",
  },
  {
    name: "Luzia e Marcos Vinicius Advocacia Ltda",
    cnpj: "24651699000118",
    registerState: "641142046131",
    dateOpen: "07/07/2018",
    site: "www.luziaemarcosviniciusadvocacialtda.com.br",
    email: "treinamento@luziaemarcosviniciusadvocacialtda.com.br",
    cep: "03472050",
    street: "Rua Manuel Figueiredo",
    number: "315",
    neighborhood: "Jardim Vila Formosa",
    city: "São Paulo",
    state: "SP",
    phone: "11996585490",
  },
  {
    name: "Mateus e Rayssa Financeira Ltda",
    cnpj: "03701720000172",
    registerState: "836859978542",
    dateOpen: "26/06/2018",
    site: "www.mateuserayssafinanceiraltda.com.br",
    email: "comunicacoes@mateuserayssafinanceiraltda.com.br",
    cep: "07176250",
    street: "Rua Soledade de Minas",
    number: "387",
    neighborhood: "Vila Nova Bonsucesso",
    city: "Guarulhos",
    state: "SP",
    phone: "11995655467",
  },
  {
    name: "Sueli e Thiago Locações de Automóveis ME",
    cnpj: "14077508000100",
    registerState: "555942545136",
    dateOpen: "22/02/2018",
    site: "www.sueliethiagolocacoesdeautomoveisme.com.br",
    email: "sac@sueliethiagolocacoesdeautomoveisme.com.br",
    cep: "17055836",
    street: "Avenida Caio Márcio Ranieri Colombo",
    number: "868",
    neighborhood: "Quinta Ranieri",
    city: "Bauru",
    state: "SP",
    phone: "14996116494",
  },
  {
    name: "Eloá e Paulo Adega ME",
    cnpj: "91541607000134",
    registerState: "929582210331",
    dateOpen: "19/08/2018",
    site: "www.eloaepauloadegame.com.br",
    email: "presidencia@eloaepauloadegame.com.br",
    cep: "13057175",
    street: "Rua Três",
    number: "196",
    neighborhood: "Loteamento Residencial Rosário",
    city: "Campinas",
    state: "SP",
    phone: "19997786094",
  },
  {
    name: "Antônia e Thiago Financeira Ltda",
    cnpj: "64671176000158",
    registerState: "848542690575",
    dateOpen: "17/06/2018",
    site: "www.antoniaethiagofinanceiraltda.com.br",
    email: "compras@antoniaethiagofinanceiraltda.com.br",
    cep: "04709060",
    street: "Rua Paulo Gomes",
    number: "973",
    neighborhood: "Vila São Francisco (Zona Sul)",
    city: "São Paulo",
    state: "SP",
    phone: "11986519122",
  },
  {
    name: "Simone e Emily Pizzaria Delivery Ltda",
    cnpj: "93619894000183",
    registerState: "127181470123",
    dateOpen: "12/02/2018",
    site: "www.simoneeemilypizzariadeliveryltda.com.br",
    email: "diretoria@simoneeemilypizzariadeliveryltda.com.br",
    cep: "08265250",
    street: "Rua Cinza das Horas",
    number: "374",
    neighborhood: "Gleba do Pêssego",
    city: "São Paulo",
    state: "SP",
    phone: "11997152858",
  },
  {
    name: "Alessandra e Benjamin Gráfica Ltda",
    cnpj: "44695420000105",
    registerState: "450339770594",
    dateOpen: "10/05/2018",
    site: "www.alessandraebenjamingraficaltda.com.br",
    email: "cobranca@alessandraebenjamingraficaltda.com.br",
    cep: "18116590",
    street: "Rua Luiz Ayala Rodrigues",
    number: "168",
    neighborhood: "Jardim Clarice I",
    city: "Votorantim",
    state: "SP",
    phone: "15981120391",
  },
  {
    name: "Renato e Ayla Telecomunicações ME",
    cnpj: "44092944000101",
    registerState: "705056696042",
    dateOpen: "24/05/2018",
    site: "www.renatoeaylatelecomunicacoesme.com.br",
    email: "marketing@renatoeaylatelecomunicacoesme.com.br",
    cep: "18700020",
    street: "Rua Maranhão",
    number: "165",
    neighborhood: "Centro",
    city: "Avaré",
    state: "SP",
    phone: "14993700507",
  },
  {
    name: "Diego e Vinicius Telecomunicações Ltda",
    cnpj: "65775119000181",
    registerState: "460326458234",
    dateOpen: "23/12/2018",
    site: "www.diegoeviniciustelecomunicacoesltda.com.br",
    email: "producao@diegoeviniciustelecomunicacoesltda.com.br",
    cep: "13073090",
    street: "Rua Votorantim",
    number: "862",
    neighborhood: "Vila Nova",
    city: "Campinas",
    state: "SP",
    phone: "19985561176",
  },
  {
    name: "Edson e Lívia Marcenaria ME",
    cnpj: "64596120000186",
    registerState: "082352500624",
    dateOpen: "18/06/2018",
    site: "www.edsoneliviamarcenariame.com.br",
    email: "seguranca@edsoneliviamarcenariame.com.br",
    cep: "19807593",
    street: "Rua Joaquim Cruz",
    number: "662",
    neighborhood: "Vila Progresso",
    city: "Assis",
    state: "SP",
    phone: "18992577783",
  },
  {
    name: "Pietro e Martin Adega Ltda",
    cnpj: "04734683000161",
    registerState: "541550589818",
    dateOpen: "18/01/2018",
    site: "www.pietroemartinadegaltda.com.br",
    email: "producao@pietroemartinadegaltda.com.br",
    cep: "13876825",
    street: "Rua João Batista Telini",
    number: "948",
    neighborhood: "Jardim das Azaléias",
    city: "São João da Boa Vista",
    state: "SP",
    phone: "19999632099",
  },
  {
    name: "Thales e Aurora Casa Noturna ME",
    cnpj: "27064958000100",
    registerState: "285050506980",
    dateOpen: "03/09/2018",
    site: "www.thaleseauroracasanoturname.com.br",
    email: "auditoria@thaleseauroracasanoturname.com.br",
    cep: "13203412",
    street: "Rua Iracy Ferreira Bueno",
    number: "224",
    neighborhood: "Vila Didi",
    city: "Jundiaí",
    state: "SP",
    phone: "11983008592",
  },
  {
    name: "Rayssa e Nicole Comercio de Bebidas Ltda",
    cnpj: "26304753000184",
    registerState: "446472371505",
    dateOpen: "02/09/2018",
    site: "www.rayssaenicolecomerciodebebidasltda.com.br",
    email: "ti@rayssaenicolecomerciodebebidasltda.com.br",
    cep: "08111560",
    street: "Rua Dorival Gomes",
    number: "273",
    neighborhood: "Jardim das Oliveiras",
    city: "São Paulo",
    state: "SP",
    phone: "11994757416",
  },
  {
    name: "Carolina e Vera Mudanças Ltda",
    cnpj: "51664691000169",
    registerState: "041859585124",
    dateOpen: "27/02/2018",
    site: "www.carolinaeveramudancasltda.com.br",
    email: "posvenda@carolinaeveramudancasltda.com.br",
    cep: "14094116",
    street: "Avenida Alfredo Ravaneli",
    number: "257",
    neighborhood: "Jardim Helena",
    city: "Ribeirão Preto",
    state: "SP",
    phone: "16994296901",
  },
  {
    name: "Carla e Alice Construções Ltda",
    cnpj: "58620830000100",
    registerState: "999546700448",
    dateOpen: "09/12/2018",
    site: "www.carlaealiceconstrucoesltda.com.br",
    email: "estoque@carlaealiceconstrucoesltda.com.br",
    cep: "04407100",
    street: "Rua Dom João Soares Coelho",
    number: "845",
    neighborhood: "Vila Império",
    city: "São Paulo",
    state: "SP",
    phone: "11988003951",
  },
  {
    name: "Eduardo e Giovanni Fotografias ME",
    cnpj: "23914128000166",
    registerState: "029171921222",
    dateOpen: "10/06/2018",
    site: "www.eduardoegiovannifotografiasme.com.br",
    email: "manutencao@eduardoegiovannifotografiasme.com.br",
    cep: "05752520",
    street: "Rua Joaquim de Magalhães",
    number: "753",
    neighborhood: "Jardim Maria Duarte",
    city: "São Paulo",
    state: "SP",
    phone: "11998016244",
  },
  {
    name: "Manuela e Nelson Telecomunicações ME",
    cnpj: "91197169000139",
    registerState: "814756703281",
    dateOpen: "26/07/2018",
    site: "www.manuelaenelsontelecomunicacoesme.com.br",
    email: "cobranca@manuelaenelsontelecomunicacoesme.com.br",
    cep: "08593410",
    street: "Rua Lontras",
    number: "183",
    neighborhood: "Chácara Águas da Pedra",
    city: "Itaquaquecetuba",
    state: "SP",
    phone: "11991457390",
  },
  {
    name: "Mariah e Clarice Vidros Ltda",
    cnpj: "49832705000164",
    registerState: "962083316211",
    dateOpen: "26/01/2018",
    site: "www.mariaheclaricevidrosltda.com.br",
    email: "cobranca@mariaheclaricevidrosltda.com.br",
    cep: "13454333",
    street: "Avenida Tenente João Benedito Caetano",
    number: "404",
    neighborhood: "Loteamento Planalto do Sol",
    city: "Santa Bárbara D'Oeste",
    state: "SP",
    phone: "19989843078",
  },
  {
    name: "Roberto e Cauê Pães e Doces Ltda",
    cnpj: "95090694000101",
    registerState: "888731147429",
    dateOpen: "17/10/2018",
    site: "www.robertoecauepaesedocesltda.com.br",
    email: "representantes@robertoecauepaesedocesltda.com.br",
    cep: "11013510",
    street: "Rua Henrique Porchat",
    number: "704",
    neighborhood: "Vila Nova",
    city: "Santos",
    state: "SP",
    phone: "13984165272",
  },
  {
    name: "Marcela e Laís Entulhos Ltda",
    cnpj: "36147094000173",
    registerState: "622284178248",
    dateOpen: "21/09/2018",
    site: "www.marcelaelaisentulhosltda.com.br",
    email: "desenvolvimento@marcelaelaisentulhosltda.com.br",
    cep: "02561000",
    street: "Avenida Monte Celeste",
    number: "142",
    neighborhood: "Vila Santa Maria",
    city: "São Paulo",
    state: "SP",
    phone: "11983622241",
  },
  {
    name: "Vera e Manoel Marketing Ltda",
    cnpj: "70315068000145",
    registerState: "026364630113",
    dateOpen: "14/05/2018",
    site: "www.veraemanoelmarketingltda.com.br",
    email: "qualidade@veraemanoelmarketingltda.com.br",
    cep: "13210888",
    street: "Rua Dois",
    number: "352",
    neighborhood: "Parque Morangaba",
    city: "Jundiaí",
    state: "SP",
    phone: "11997741801",
  },
  {
    name: "Mariah e Filipe Entregas Expressas ME",
    cnpj: "04683785000103",
    registerState: "299860230811",
    dateOpen: "06/01/2018",
    site: "www.mariahefilipeentregasexpressasme.com.br",
    email: "qualidade@mariahefilipeentregasexpressasme.com.br",
    cep: "13420168",
    street: "Rua Frutuoso Viana",
    number: "601",
    neighborhood: "Parque Santa Cecília",
    city: "Piracicaba",
    state: "SP",
    phone: "19998998101",
  },
  {
    name: "Arthur e Nelson Advocacia ME",
    cnpj: "23747956000157",
    registerState: "627751410474",
    dateOpen: "23/06/2018",
    site: "www.arthurenelsonadvocaciame.com.br",
    email: "orcamento@arthurenelsonadvocaciame.com.br",
    cep: "14700901",
    street: "Praça Barão do Rio Branco 48",
    number: "786",
    neighborhood: "Centro",
    city: "Bebedouro",
    state: "SP",
    phone: "17988092469",
  },
  {
    name: "Alexandre e Mariah Pães e Doces Ltda",
    cnpj: "40925100000143",
    registerState: "677509534688",
    dateOpen: "14/03/2018",
    site: "www.alexandreemariahpaesedocesltda.com.br",
    email: "comunicacoes@alexandreemariahpaesedocesltda.com.br",
    cep: "13411080",
    street: "Rua Corumbataí",
    number: "221",
    neighborhood: "Santa Terezinha",
    city: "Piracicaba",
    state: "SP",
    phone: "19992176730",
  },
  {
    name: "Elisa e Marcos Assessoria Jurídica Ltda",
    cnpj: "49609285000151",
    registerState: "739098709557",
    dateOpen: "13/10/2018",
    site: "www.elisaemarcosassessoriajuridicaltda.com.br",
    email: "faleconosco@elisaemarcosassessoriajuridicaltda.com.br",
    cep: "17011225",
    street: "Rua Nove",
    number: "721",
    neighborhood: "Vila Maracy",
    city: "Bauru",
    state: "SP",
    phone: "14981476389",
  },
  {
    name: "Bento e Evelyn Buffet Ltda",
    cnpj: "32069090000163",
    registerState: "066505458520",
    dateOpen: "23/05/2018",
    site: "www.bentoeevelynbuffetltda.com.br",
    email: "fabricacao@bentoeevelynbuffetltda.com.br",
    cep: "05187919",
    street: "Rua Gregório Tagle 301",
    number: "166",
    neighborhood: "Jardim Ipanema (Zona Oeste)",
    city: "São Paulo",
    state: "SP",
    phone: "11991085159",
  },
  {
    name: "Helena e Leonardo Marketing Ltda",
    cnpj: "32390036000115",
    registerState: "076746820534",
    dateOpen: "08/12/2018",
    site: "www.helenaeleonardomarketingltda.com.br",
    email: "pesquisa@helenaeleonardomarketingltda.com.br",
    cep: "14811068",
    street: "Rua Minas Gerais",
    number: "530",
    neighborhood: "Jardim Silvânia (Vila Xavier)",
    city: "Araraquara",
    state: "SP",
    phone: "16997962084",
  },
  {
    name: "Josefa e Carolina Construções ME",
    cnpj: "89968689000156",
    registerState: "061833155660",
    dateOpen: "08/12/2018",
    site: "www.josefaecarolinaconstrucoesme.com.br",
    email: "financeiro@josefaecarolinaconstrucoesme.com.br",
    cep: "14808329",
    street: "Rua Doutor Moacyr Porto",
    number: "288",
    neighborhood: "Jardim Residencial Silvestre",
    city: "Araraquara",
    state: "SP",
    phone: "16998387632",
  },
  {
    name: "Pedro Henrique e Emanuelly Assessoria Jurídica Ltda",
    cnpj: "34256299000161",
    registerState: "956884134475",
    dateOpen: "14/09/2018",
    site: "www.pedrohenriqueeemanuellyassessoriajuridicaltda.com.br",
    email: "posvenda@pedrohenriqueeemanuellyassessoriajuridicaltda.com.br",
    cep: "13185053",
    street: "Rua Joel Cardoso de Sá",
    number: "560",
    neighborhood: "Jardim Santa Emília",
    city: "Hortolândia",
    state: "SP",
    phone: "19983818985",
  },
  {
    name: "Marcos e Vinicius Comercio de Bebidas Ltda",
    cnpj: "36565183000130",
    registerState: "445108392306",
    dateOpen: "16/02/2018",
    site: "www.marcoseviniciuscomerciodebebidasltda.com.br",
    email: "rh@marcoseviniciuscomerciodebebidasltda.com.br",
    cep: "14169260",
    street: "Rua Elis Regina Carvalho Costa",
    number: "190",
    neighborhood: "Jardim Canaã",
    city: "Sertãozinho",
    state: "SP",
    phone: "16982642241",
  },
  {
    name: "Lucas e Nicole Mudanças Ltda",
    cnpj: "10496089000182",
    registerState: "137792978483",
    dateOpen: "15/04/2018",
    site: "www.lucasenicolemudancasltda.com.br",
    email: "estoque@lucasenicolemudancasltda.com.br",
    cep: "05283060",
    street: "Rua Santo Expedito",
    number: "967",
    neighborhood: "Residencial Sol Nascente",
    city: "São Paulo",
    state: "SP",
    phone: "11982189329",
  },
  {
    name: "Tânia e Yuri Assessoria Jurídica ME",
    cnpj: "58389605000104",
    registerState: "053040099230",
    dateOpen: "09/02/2018",
    site: "www.taniaeyuriassessoriajuridicame.com.br",
    email: "contabilidade@taniaeyuriassessoriajuridicame.com.br",
    cep: "17602848",
    street: "Rua José Venâncio de Oliveira",
    number: "914",
    neighborhood: "Conjunto Habitacional Antônio Pereira Gaspar",
    city: "Tupã",
    state: "SP",
    phone: "14983250851",
  },
  {
    name: "Ian e Pedro Advocacia Ltda",
    cnpj: "33206928000186",
    registerState: "098455447341",
    dateOpen: "27/12/2018",
    site: "www.ianepedroadvocacialtda.com.br",
    email: "treinamento@ianepedroadvocacialtda.com.br",
    cep: "02945060",
    street: "Rua Mathilde Carlos Montesanti",
    number: "745",
    neighborhood: "Jardim Cidade Pirituba",
    city: "São Paulo",
    state: "SP",
    phone: "11983871018",
  },
  {
    name: "Heloise e Mariana Lavanderia ME",
    cnpj: "26087976000137",
    registerState: "969224848248",
    dateOpen: "21/12/2018",
    site: "www.heloiseemarianalavanderiame.com.br",
    email: "qualidade@heloiseemarianalavanderiame.com.br",
    cep: "09980250",
    street: "Rua Maria Rita Pereira",
    number: "359",
    neighborhood: "Serraria",
    city: "Diadema",
    state: "SP",
    phone: "11991841520",
  },
  {
    name: "Felipe e Benedita Vidros ME",
    cnpj: "28949730000152",
    registerState: "110101130617",
    dateOpen: "08/05/2018",
    site: "www.felipeebeneditavidrosme.com.br",
    email: "rh@felipeebeneditavidrosme.com.br",
    cep: "11711400",
    street: "Rua Vinte e Sete",
    number: "792",
    neighborhood: "Imperador",
    city: "Praia Grande",
    state: "SP",
    phone: "13983586651",
  },
  {
    name: "Eduardo e Marlene Ferragens Ltda",
    cnpj: "90277638000167",
    registerState: "093432314445",
    dateOpen: "18/07/2018",
    site: "www.eduardoemarleneferragensltda.com.br",
    email: "contato@eduardoemarleneferragensltda.com.br",
    cep: "09371480",
    street: "Rua João Brancalion",
    number: "651",
    neighborhood: "Parque São Vicente",
    city: "Mauá",
    state: "SP",
    phone: "11987370094",
  },
  {
    name: "Luna e Iago Doces & Salgados ME",
    cnpj: "19746165000135",
    registerState: "284925420379",
    dateOpen: "02/03/2018",
    site: "www.lunaeiagodocessalgadosme.com.br",
    email: "presidencia@lunaeiagodocessalgadosme.com.br",
    cep: "03978600",
    street: "Rua Paulo Leminski",
    number: "976",
    neighborhood: "Conjunto Residencial Sitio Oratório",
    city: "São Paulo",
    state: "SP",
    phone: "11985357537",
  },
  {
    name: "Otávio e Simone Assessoria Jurídica ME",
    cnpj: "46692948000156",
    registerState: "462418971328",
    dateOpen: "24/01/2018",
    site: "www.otavioesimoneassessoriajuridicame.com.br",
    email: "fiscal@otavioesimoneassessoriajuridicame.com.br",
    cep: "09360525",
    street: "Rua Benedito José de Godoy",
    number: "131",
    neighborhood: "Jardim Anchieta",
    city: "Mauá",
    state: "SP",
    phone: "11982091404",
  },
  {
    name: "Antonio e Tânia Lavanderia Ltda",
    cnpj: "20439538000104",
    registerState: "747055191782",
    dateOpen: "19/06/2018",
    site: "www.antonioetanialavanderialtda.com.br",
    email: "faleconosco@antonioetanialavanderialtda.com.br",
    cep: "13465438",
    street: "Praça Oswaldo Salgals",
    number: "603",
    neighborhood: "Chácara Rodrigues",
    city: "Americana",
    state: "SP",
    phone: "19998544072",
  },
  {
    name: "Lorena e Luís Telecom Ltda",
    cnpj: "88646806000100",
    registerState: "789628108686",
    dateOpen: "23/04/2018",
    site: "www.lorenaeluistelecomltda.com.br",
    email: "seguranca@lorenaeluistelecomltda.com.br",
    cep: "14096310",
    street: "Rua Felipe Miguel Curi",
    number: "371",
    neighborhood: "Ribeirânia",
    city: "Ribeirão Preto",
    state: "SP",
    phone: "16982160641",
  },
  {
    name: "Benedita e Igor Alimentos Ltda",
    cnpj: "72166081000105",
    registerState: "586612732323",
    dateOpen: "27/06/2018",
    site: "www.beneditaeigoralimentosltda.com.br",
    email: "fabricacao@beneditaeigoralimentosltda.com.br",
    cep: "18046141",
    street: "Rua Lituânia",
    number: "151",
    neighborhood: "Jardim Pagliato",
    city: "Sorocaba",
    state: "SP",
    phone: "15986546798",
  },
  {
    name: "Miguel e Allana Limpeza ME",
    cnpj: "81985304000157",
    registerState: "574019679595",
    dateOpen: "11/04/2018",
    site: "www.migueleallanalimpezame.com.br",
    email: "contato@migueleallanalimpezame.com.br",
    cep: "04771200",
    street: "Rua Morais Torres",
    number: "444",
    neighborhood: "Jardim dos Lagos",
    city: "São Paulo",
    state: "SP",
    phone: "11984820065",
  },
  {
    name: "Caroline e Gael Filmagens Ltda",
    cnpj: "29784857000121",
    registerState: "125719932831",
    dateOpen: "26/07/2018",
    site: "www.carolineegaelfilmagensltda.com.br",
    email: "administracao@carolineegaelfilmagensltda.com.br",
    cep: "06833083",
    street: "Rua Joseph Celano",
    number: "890",
    neighborhood: "Jardim Magaly",
    city: "Embu das Artes",
    state: "SP",
    phone: "11992695922",
  },
  {
    name: "Jéssica e Renan Contábil Ltda",
    cnpj: "06308870000109",
    registerState: "035037587986",
    dateOpen: "23/06/2018",
    site: "www.jessicaerenancontabilltda.com.br",
    email: "faleconosco@jessicaerenancontabilltda.com.br",
    cep: "02635000",
    street: "Rua Pedra Bonita",
    number: "706",
    neighborhood: "Vila Pedra Branca",
    city: "São Paulo",
    state: "SP",
    phone: "11985819414",
  },
  {
    name: "Gabrielly e Joana Joalheria ME",
    cnpj: "60966427000170",
    registerState: "557143339377",
    dateOpen: "24/09/2018",
    site: "www.gabriellyejoanajoalheriame.com.br",
    email: "producao@gabriellyejoanajoalheriame.com.br",
    cep: "09842100",
    street: "Estrada da Cama Patente",
    number: "911",
    neighborhood: "Alvarenga",
    city: "São Bernardo do Campo",
    state: "SP",
    phone: "11994323016",
  },
  {
    name: "Joaquim e Samuel Contábil ME",
    cnpj: "80005429000100",
    registerState: "065161365410",
    dateOpen: "23/07/2018",
    site: "www.joaquimesamuelcontabilme.com.br",
    email: "atendimento@joaquimesamuelcontabilme.com.br",
    cep: "09272439",
    street: "Travessa Setiba",
    number: "383",
    neighborhood: "Jardim Alzira Franco",
    city: "Santo André",
    state: "SP",
    phone: "11997375737",
  },
  {
    name: "Alexandre e Giovana Telecom Ltda",
    cnpj: "40478449000184",
    registerState: "806079654250",
    dateOpen: "08/07/2018",
    site: "www.alexandreegiovanatelecomltda.com.br",
    email: "representantes@alexandreegiovanatelecomltda.com.br",
    cep: "13416421",
    street: "Rua Doutor João Sampaio",
    number: "250",
    neighborhood: "Cidade Jardim",
    city: "Piracicaba",
    state: "SP",
    phone: "19982331714",
  },
  {
    name: "Mariane e Bárbara Buffet ME",
    cnpj: "17323589000161",
    registerState: "179777197660",
    dateOpen: "11/02/2018",
    site: "www.marianeebarbarabuffetme.com.br",
    email: "desenvolvimento@marianeebarbarabuffetme.com.br",
    cep: "14409621",
    street: "Rua José Liboni",
    number: "297",
    neighborhood: "City Petrópolis",
    city: "Franca",
    state: "SP",
    phone: "16992077319",
  },
  {
    name: "Ester e Sophia Comercio de Bebidas ME",
    cnpj: "75556252000109",
    registerState: "517020529430",
    dateOpen: "16/08/2018",
    site: "www.esteresophiacomerciodebebidasme.com.br",
    email: "suporte@esteresophiacomerciodebebidasme.com.br",
    cep: "07155390",
    street: "Rua Carnaubal",
    number: "211",
    neighborhood: "Cidade Seródio",
    city: "Guarulhos",
    state: "SP",
    phone: "11985599037",
  },
  {
    name: "Gabriela e Marcela Advocacia ME",
    cnpj: "57104909000107",
    registerState: "394168793715",
    dateOpen: "02/03/2018",
    site: "www.gabrielaemarcelaadvocaciame.com.br",
    email: "manutencao@gabrielaemarcelaadvocaciame.com.br",
    cep: "14166220",
    street: "Rua Salvador Aprile",
    number: "936",
    neighborhood: "Jardim Paraíso",
    city: "Sertãozinho",
    state: "SP",
    phone: "16982452463",
  },
  {
    name: "Thiago e Elisa Alimentos Ltda",
    cnpj: "06941241000111",
    registerState: "157352050336",
    dateOpen: "10/10/2018",
    site: "www.thiagoeelisaalimentosltda.com.br",
    email: "fabricacao@thiagoeelisaalimentosltda.com.br",
    cep: "12608240",
    street: "Rua Tenente Frederico Ferreira",
    number: "991",
    neighborhood: "Vila Hepacare",
    city: "Lorena",
    state: "SP",
    phone: "12986421573",
  },
  {
    name: "Márcia e Calebe Mudanças Ltda",
    cnpj: "26908511000108",
    registerState: "023295568538",
    dateOpen: "23/01/2018",
    site: "www.marciaecalebemudancasltda.com.br",
    email: "estoque@marciaecalebemudancasltda.com.br",
    cep: "17209318",
    street: "Rua José Antoniolli",
    number: "294",
    neighborhood: "Jardim Cila de Lúcio Bauab",
    city: "Jaú",
    state: "SP",
    phone: "14993448614",
  },
  {
    name: "Roberto e Nicole Pizzaria Delivery ME",
    cnpj: "98185910000190",
    registerState: "150044378004",
    dateOpen: "25/01/2018",
    site: "www.robertoenicolepizzariadeliveryme.com.br",
    email: "representantes@robertoenicolepizzariadeliveryme.com.br",
    cep: "16203193",
    street: "Rua Antônio Fabrício",
    number: "737",
    neighborhood: "Nucleo Habitacional Ivone Alves Palma",
    city: "Birigüi",
    state: "SP",
    phone: "18987582313",
  },
  {
    name: "Raul e Carla Ferragens Ltda",
    cnpj: "47722023000173",
    registerState: "231658988955",
    dateOpen: "22/11/2018",
    site: "www.raulecarlaferragensltda.com.br",
    email: "auditoria@raulecarlaferragensltda.com.br",
    cep: "13276040",
    street: "Avenida Guarani",
    number: "643",
    neighborhood: "Vila Coqueiro",
    city: "Valinhos",
    state: "SP",
    phone: "19982815323",
  },
  {
    name: "Arthur e Oliver Doces & Salgados Ltda",
    cnpj: "07059618000176",
    registerState: "444694697806",
    dateOpen: "18/03/2018",
    site: "www.arthureoliverdocessalgadosltda.com.br",
    email: "financeiro@arthureoliverdocessalgadosltda.com.br",
    cep: "13630502",
    street: "Rua Otto Schubart",
    number: "156",
    neighborhood: "Jardim Eldorado",
    city: "Pirassununga",
    state: "SP",
    phone: "19991207180",
  },
  {
    name: "Adriana e Aline Comercio de Bebidas Ltda",
    cnpj: "58693406000187",
    registerState: "383256818952",
    dateOpen: "13/09/2018",
    site: "www.adrianaealinecomerciodebebidasltda.com.br",
    email: "ouvidoria@adrianaealinecomerciodebebidasltda.com.br",
    cep: "09341420",
    street: "Rua Saturnino Juvenal Lima",
    number: "518",
    neighborhood: "Jardim Maria Eneida",
    city: "Mauá",
    state: "SP",
    phone: "11981314451",
  },
  {
    name: "Nathan e Levi Advocacia ME",
    cnpj: "96866400000153",
    registerState: "695204314950",
    dateOpen: "17/11/2018",
    site: "www.nathaneleviadvocaciame.com.br",
    email: "fiscal@nathaneleviadvocaciame.com.br",
    cep: "06654205",
    street: "Viela Santa Berenice",
    number: "269",
    neighborhood: "Vila Doutor Cardoso",
    city: "Itapevi",
    state: "SP",
    phone: "11989636762",
  },
  {
    name: "Rita e André Consultoria Financeira ME",
    cnpj: "85253164000183",
    registerState: "502806920508",
    dateOpen: "11/08/2018",
    site: "www.ritaeandreconsultoriafinanceirame.com.br",
    email: "producao@ritaeandreconsultoriafinanceirame.com.br",
    cep: "14030110",
    street: "Rua Pierino Malardo",
    number: "272",
    neighborhood: "Vila Virgínia",
    city: "Ribeirão Preto",
    state: "SP",
    phone: "16992839900",
  },
  {
    name: "Arthur e Alícia Construções Ltda",
    cnpj: "51111107000148",
    registerState: "721678829496",
    dateOpen: "25/10/2018",
    site: "www.arthurealiciaconstrucoesltda.com.br",
    email: "seguranca@arthurealiciaconstrucoesltda.com.br",
    cep: "13631600",
    street: "Rua José Dioguinho Baldovinoti",
    number: "727",
    neighborhood: "Jardim Brasília",
    city: "Pirassununga",
    state: "SP",
    phone: "19994419540",
  },
  {
    name: "Joana e Eduardo Joalheria ME",
    cnpj: "08547861000104",
    registerState: "403576919571",
    dateOpen: "19/02/2018",
    site: "www.joanaeeduardojoalheriame.com.br",
    email: "marketing@joanaeeduardojoalheriame.com.br",
    cep: "13604121",
    street: "Rua Ângelo Bertolini",
    number: "272",
    neighborhood: "Jardim Alto das Araras",
    city: "Araras",
    state: "SP",
    phone: "19982541016",
  },
  {
    name: "Luís e Roberto Eletrônica ME",
    cnpj: "10135879000132",
    registerState: "314621910826",
    dateOpen: "08/04/2018",
    site: "www.luiserobertoeletronicame.com.br",
    email: "juridico@luiserobertoeletronicame.com.br",
    cep: "13321000",
    street: "Rua Marrey Júnior",
    number: "953",
    neighborhood: "Jardim Maria José",
    city: "Salto",
    state: "SP",
    phone: "11994797737",
  },
  {
    name: "Emily e Lorena Telecom ME",
    cnpj: "10883847000115",
    registerState: "775510280180",
    dateOpen: "24/06/2018",
    site: "www.emilyelorenatelecomme.com.br",
    email: "fabricacao@emilyelorenatelecomme.com.br",
    cep: "13270080",
    street: "Rua Eugênio Franceschini",
    number: "934",
    neighborhood: "Centro",
    city: "Valinhos",
    state: "SP",
    phone: "19991917692",
  },
  {
    name: "Martin e Bernardo Limpeza ME",
    cnpj: "61162711000157",
    registerState: "680000693705",
    dateOpen: "14/04/2018",
    site: "www.martinebernardolimpezame.com.br",
    email: "contato@martinebernardolimpezame.com.br",
    cep: "06804440",
    street: "Rua dos Artistas",
    number: "363",
    neighborhood: "Jardim Recanto da Fonte",
    city: "Embu das Artes",
    state: "SP",
    phone: "11982449213",
  },
  {
    name: "Heloise e Jaqueline Marketing ME",
    cnpj: "99588542000194",
    registerState: "163164127563",
    dateOpen: "23/05/2018",
    site: "www.heloiseejaquelinemarketingme.com.br",
    email: "seguranca@heloiseejaquelinemarketingme.com.br",
    cep: "03178090",
    street: "Rua Arabutá",
    number: "804",
    neighborhood: "Alto da Mooca",
    city: "São Paulo",
    state: "SP",
    phone: "11988445430",
  },
  {
    name: "Gael e Thomas Pizzaria Ltda",
    cnpj: "47017007000180",
    registerState: "899139670993",
    dateOpen: "12/07/2018",
    site: "www.gaelethomaspizzarialtda.com.br",
    email: "diretoria@gaelethomaspizzarialtda.com.br",
    cep: "05551210",
    street: "Rua Bernardo Teles",
    number: "528",
    neighborhood: "Jardim Raposo Tavares",
    city: "São Paulo",
    state: "SP",
    phone: "11989541300",
  },
  {
    name: "Adriana e Laura Telecomunicações ME",
    cnpj: "17274027000175",
    registerState: "919794890220",
    dateOpen: "20/08/2018",
    site: "www.adrianaelauratelecomunicacoesme.com.br",
    email: "diretoria@adrianaelauratelecomunicacoesme.com.br",
    cep: "04814100",
    street: "Rua Gaspar José Raposo",
    number: "330",
    neighborhood: "Jardim Maria Rita",
    city: "São Paulo",
    state: "SP",
    phone: "11985653206",
  },
  {
    name: "Heloisa e Nicole Marketing ME",
    cnpj: "31123155000149",
    registerState: "337020071677",
    dateOpen: "22/03/2018",
    site: "www.heloisaenicolemarketingme.com.br",
    email: "treinamento@heloisaenicolemarketingme.com.br",
    cep: "07082090",
    street: "Rua Coronel Freitas",
    number: "343",
    neighborhood: "Jardim Terezópolis",
    city: "Guarulhos",
    state: "SP",
    phone: "11986279458",
  },
  {
    name: "Sueli e Pedro Joalheria ME",
    cnpj: "59041921000145",
    registerState: "939743173110",
    dateOpen: "09/05/2018",
    site: "www.sueliepedrojoalheriame.com.br",
    email: "tesouraria@sueliepedrojoalheriame.com.br",
    cep: "13486008",
    street: "Rua Ariovaldo de Oliveira",
    number: "644",
    neighborhood: "Jardim São Pedro",
    city: "Limeira",
    state: "SP",
    phone: "19999043930",
  },
  {
    name: "Márcio e Benedito Alimentos Ltda",
    cnpj: "79937810000129",
    registerState: "498677188249",
    dateOpen: "06/05/2018",
    site: "www.marcioebeneditoalimentosltda.com.br",
    email: "suporte@marcioebeneditoalimentosltda.com.br",
    cep: "13506107",
    street: "Rua 9 JV",
    number: "341",
    neighborhood: "Jardim Vilage",
    city: "Rio Claro",
    state: "SP",
    phone: "19983496963",
  },
  {
    name: "Pietra e Nathan Filmagens ME",
    cnpj: "44592374000119",
    registerState: "237396674944",
    dateOpen: "08/08/2018",
    site: "www.pietraenathanfilmagensme.com.br",
    email: "representantes@pietraenathanfilmagensme.com.br",
    cep: "15055420",
    street: "Rua Elias Abissamra",
    number: "268",
    neighborhood: "Jardim Bordon",
    city: "São José do Rio Preto",
    state: "SP",
    phone: "17994845161",
  },
  {
    name: "Marli e Sérgio Fotografias Ltda",
    cnpj: "83304067000147",
    registerState: "584919895080",
    dateOpen: "23/02/2018",
    site: "www.marliesergiofotografiasltda.com.br",
    email: "fabricacao@marliesergiofotografiasltda.com.br",
    cep: "13202543",
    street: "Rua Capote Valente",
    number: "636",
    neighborhood: "Vila Arens II",
    city: "Jundiaí",
    state: "SP",
    phone: "11994937064",
  },
  {
    name: "Isis e Bryan Joalheria ME",
    cnpj: "21241517000142",
    registerState: "200511888801",
    dateOpen: "17/12/2018",
    site: "www.isisebryanjoalheriame.com.br",
    email: "posvenda@isisebryanjoalheriame.com.br",
    cep: "16055514",
    street: "Rua A",
    number: "589",
    neighborhood: "Aviação",
    city: "Araçatuba",
    state: "SP",
    phone: "18993980432",
  },
  {
    name: "Gael e Manuel Consultoria Financeira ME",
    cnpj: "92431077000134",
    registerState: "076398859069",
    dateOpen: "15/11/2018",
    site: "www.gaelemanuelconsultoriafinanceirame.com.br",
    email: "ti@gaelemanuelconsultoriafinanceirame.com.br",
    cep: "03717038",
    street: "Viela Madruga",
    number: "660",
    neighborhood: "Jardim Piratininga",
    city: "São Paulo",
    state: "SP",
    phone: "11992508613",
  },
  {
    name: "Brenda e Emanuelly Alimentos Ltda",
    cnpj: "95005769000109",
    registerState: "303395620680",
    dateOpen: "23/01/2018",
    site: "www.brendaeemanuellyalimentosltda.com.br",
    email: "fabricacao@brendaeemanuellyalimentosltda.com.br",
    cep: "12308250",
    street: "Rua Cândido Pires de Almeida",
    number: "117",
    neighborhood: "Centro",
    city: "Jacareí",
    state: "SP",
    phone: "12986781577",
  },
  {
    name: "Renata e Pietro Esportes ME",
    cnpj: "39258767000150",
    registerState: "548794395525",
    dateOpen: "18/11/2018",
    site: "www.renataepietroesportesme.com.br",
    email: "producao@renataepietroesportesme.com.br",
    cep: "08715130",
    street: "Rua Adriano Francisco Salgado",
    number: "436",
    neighborhood: "Vila Sud Menuci",
    city: "Mogi das Cruzes",
    state: "SP",
    phone: "11982055854",
  },
  {
    name: "Milena e Sebastião Comercio de Bebidas Ltda",
    cnpj: "25576944000132",
    registerState: "272800307223",
    dateOpen: "16/10/2018",
    site: "www.milenaesebastiaocomerciodebebidasltda.com.br",
    email: "contabilidade@milenaesebastiaocomerciodebebidasltda.com.br",
    cep: "15057506",
    street: "Rua Arlindo Valente",
    number: "638",
    neighborhood: "Parque São Miguel",
    city: "São José do Rio Preto",
    state: "SP",
    phone: "17983785641",
  },
  {
    name: "Liz e Regina Comercio de Bebidas Ltda",
    cnpj: "44175470000161",
    registerState: "818926621400",
    dateOpen: "17/04/2018",
    site: "www.lizereginacomerciodebebidasltda.com.br",
    email: "almoxarifado@lizereginacomerciodebebidasltda.com.br",
    cep: "15808258",
    street: "Rua Itanhem",
    number: "226",
    neighborhood: "Loteamento Bom Pastor",
    city: "Catanduva",
    state: "SP",
    phone: "17993011845",
  },
  {
    name: "Murilo e Fábio Informática ME",
    cnpj: "04219077000108",
    registerState: "110318788440",
    dateOpen: "24/04/2018",
    site: "www.muriloefabioinformaticame.com.br",
    email: "fiscal@muriloefabioinformaticame.com.br",
    cep: "08223121",
    street: "Viela Cinco",
    number: "381",
    neighborhood: "Cidade Antônio Estevão de Carvalho",
    city: "São Paulo",
    state: "SP",
    phone: "11995557150",
  },
  {
    name: "Tomás e Vinicius Fotografias ME",
    cnpj: "23022858000152",
    registerState: "986350323232",
    dateOpen: "25/04/2018",
    site: "www.tomaseviniciusfotografiasme.com.br",
    email: "almoxarifado@tomaseviniciusfotografiasme.com.br",
    cep: "12322430",
    street: "Rua José Abrahão",
    number: "538",
    neighborhood: "São João",
    city: "Jacareí",
    state: "SP",
    phone: "12987360479",
  },
  {
    name: "Breno e Murilo Publicidade e Propaganda ME",
    cnpj: "28025662000135",
    registerState: "418825316875",
    dateOpen: "12/05/2018",
    site: "www.brenoemurilopublicidadeepropagandame.com.br",
    email: "qualidade@brenoemurilopublicidadeepropagandame.com.br",
    cep: "18605050",
    street: "Rua Frederico Michelin",
    number: "309",
    neighborhood: "Conjunto Habitacional Humberto Popolo",
    city: "Botucatu",
    state: "SP",
    phone: "14981067730",
  },
  {
    name: "Lúcia e Analu Eletrônica Ltda",
    cnpj: "21565924000105",
    registerState: "530733135960",
    dateOpen: "26/01/2018",
    site: "www.luciaeanalueletronicaltda.com.br",
    email: "presidencia@luciaeanalueletronicaltda.com.br",
    cep: "13219007",
    street: "Rua Severo Maltoni",
    number: "795",
    neighborhood: "Vila Santana II",
    city: "Jundiaí",
    state: "SP",
    phone: "11989613834",
  },
  {
    name: "Nelson e Marcos Limpeza ME",
    cnpj: "53625187000185",
    registerState: "091087113679",
    dateOpen: "17/06/2018",
    site: "www.nelsonemarcoslimpezame.com.br",
    email: "seguranca@nelsonemarcoslimpezame.com.br",
    cep: "03324090",
    street: "Rua Guatacaba",
    number: "121",
    neighborhood: "Vila Gomes Cardim",
    city: "São Paulo",
    state: "SP",
    phone: "11993877451",
  },
  {
    name: "Henry e Gustavo Ferragens Ltda",
    cnpj: "86305580000140",
    registerState: "456394346408",
    dateOpen: "15/12/2018",
    site: "www.henryegustavoferragensltda.com.br",
    email: "financeiro@henryegustavoferragensltda.com.br",
    cep: "03051000",
    street: "Rua do Hipódromo",
    number: "307",
    neighborhood: "Brás",
    city: "São Paulo",
    state: "SP",
    phone: "11994547796",
  },
  {
    name: "Martin e Betina Vidros ME",
    cnpj: "22352752000154",
    registerState: "150774942360",
    dateOpen: "03/07/2018",
    site: "www.martinebetinavidrosme.com.br",
    email: "financeiro@martinebetinavidrosme.com.br",
    cep: "05334120",
    street: "Praça Bento de Assis",
    number: "895",
    neighborhood: "Jaguaré",
    city: "São Paulo",
    state: "SP",
    phone: "11988845168",
  },
  {
    name: "Samuel e Pietro Comercio de Bebidas ME",
    cnpj: "51106635000109",
    registerState: "561105800987",
    dateOpen: "16/02/2018",
    site: "www.samuelepietrocomerciodebebidasme.com.br",
    email: "marketing@samuelepietrocomerciodebebidasme.com.br",
    cep: "17018780",
    street: "Rua Walter Pires Ramos",
    number: "999",
    neighborhood: "Residencial Villaggio II",
    city: "Bauru",
    state: "SP",
    phone: "14989969689",
  },
  {
    name: "Igor e Manuel Mudanças ME",
    cnpj: "71660190000111",
    registerState: "650140939435",
    dateOpen: "21/03/2018",
    site: "www.igoremanuelmudancasme.com.br",
    email: "orcamento@igoremanuelmudancasme.com.br",
    cep: "15808205",
    street: "Rua Antônio Frederico Ozanan",
    number: "432",
    neighborhood: "Loteamento Solo Sagrado",
    city: "Catanduva",
    state: "SP",
    phone: "17999480433",
  },
  {
    name: "Giovanni e Valentina Casa Noturna Ltda",
    cnpj: "81057337000137",
    registerState: "591354862028",
    dateOpen: "09/01/2018",
    site: "www.giovannievalentinacasanoturnaltda.com.br",
    email: "contato@giovannievalentinacasanoturnaltda.com.br",
    cep: "05362110",
    street: "Rua André Carnegie",
    number: "836",
    neighborhood: "Vila São Luís(Zona Oeste)",
    city: "São Paulo",
    state: "SP",
    phone: "11999348657",
  },
  {
    name: "Patrícia e Eduarda Doces & Salgados ME",
    cnpj: "98103810000178",
    registerState: "809052552498",
    dateOpen: "11/10/2018",
    site: "www.patriciaeeduardadocessalgadosme.com.br",
    email: "fiscal@patriciaeeduardadocessalgadosme.com.br",
    cep: "09131470",
    street: "Travessa Paulo Duarte",
    number: "923",
    neighborhood: "Condomínio Maracanã",
    city: "Santo André",
    state: "SP",
    phone: "11992724540",
  },
  {
    name: "Giovanni e Ayla Consultoria Financeira Ltda",
    cnpj: "52450009000106",
    registerState: "070393001365",
    dateOpen: "05/05/2018",
    site: "www.giovannieaylaconsultoriafinanceiraltda.com.br",
    email: "vendas@giovannieaylaconsultoriafinanceiraltda.com.br",
    cep: "19060410",
    street: "Rua José Monzoni",
    number: "376",
    neighborhood: "Jardim Petrópolis",
    city: "Presidente Prudente",
    state: "SP",
    phone: "18998745642",
  },
  {
    name: "Calebe e Yasmin Casa Noturna ME",
    cnpj: "49525469000133",
    registerState: "950245399438",
    dateOpen: "19/11/2018",
    site: "www.calebeeyasmincasanoturname.com.br",
    email: "desenvolvimento@calebeeyasmincasanoturname.com.br",
    cep: "02441001",
    street: "Avenida Ultramarino",
    number: "483",
    neighborhood: "Lauzane Paulista",
    city: "São Paulo",
    state: "SP",
    phone: "11985639895",
  },
  {
    name: "Isabela e Elias Telas Ltda",
    cnpj: "77117946000167",
    registerState: "676870368908",
    dateOpen: "04/08/2018",
    site: "www.isabelaeeliastelasltda.com.br",
    email: "qualidade@isabelaeeliastelasltda.com.br",
    cep: "03274140",
    street: "Rua Rio do Sul",
    number: "559",
    neighborhood: "Vila Santa Clara",
    city: "São Paulo",
    state: "SP",
    phone: "11997215898",
  },
  {
    name: "Lívia e Benício Ferragens Ltda",
    cnpj: "55731770000104",
    registerState: "829655058238",
    dateOpen: "10/09/2018",
    site: "www.liviaebenicioferragensltda.com.br",
    email: "financeiro@liviaebenicioferragensltda.com.br",
    cep: "04320130",
    street: "Praça Clóvis Rodrigues de Souza",
    number: "744",
    neighborhood: "Cidade Vargas",
    city: "São Paulo",
    state: "SP",
    phone: "11986313408",
  },
  {
    name: "Ryan e Elias Telecom Ltda",
    cnpj: "06268261000173",
    registerState: "047288622602",
    dateOpen: "23/11/2018",
    site: "www.ryaneeliastelecomltda.com.br",
    email: "comunicacoes@ryaneeliastelecomltda.com.br",
    cep: "14706082",
    street: "Rua Paraguai",
    number: "138",
    neighborhood: "Jardim Marajá",
    city: "Bebedouro",
    state: "SP",
    phone: "17999109033",
  },
  {
    name: "Fernando e Gabriel Padaria Ltda",
    cnpj: "04427574000100",
    registerState: "709401272700",
    dateOpen: "05/10/2018",
    site: "www.fernandoegabrielpadarialtda.com.br",
    email: "rh@fernandoegabrielpadarialtda.com.br",
    cep: "14887058",
    street: "Rua Luiz Arrobas Martins",
    number: "263",
    neighborhood: "Jardim Santa Rita",
    city: "Jaboticabal",
    state: "SP",
    phone: "16993723495",
  },
  {
    name: "Luciana e Sebastião Advocacia ME",
    cnpj: "50416749000192",
    registerState: "155490162201",
    dateOpen: "04/03/2018",
    site: "www.lucianaesebastiaoadvocaciame.com.br",
    email: "administracao@lucianaesebastiaoadvocaciame.com.br",
    cep: "06226170",
    street: "Rua Salvador",
    number: "296",
    neighborhood: "Rochdale",
    city: "Osasco",
    state: "SP",
    phone: "11984728191",
  },
];
function generateEnterprise() {
  return enterprise[getRandomInt(0, enterprise.length - 1)];
}
const cnhs = [
  "05678360072",
  "24012260817",
  "08422795918",
  "23195659970",
  "76578081346",
  "76606356466",
  "72533709838",
  "54552913433",
  "63542282183",
  "38522930940",
  "47616671089",
  "76985509105",
  "14184756289",
  "30687436133",
  "10790030996",
  "13242881078",
  "73478739430",
  "64759350394",
  "21518717101",
  "39555543295",
  "34365997315",
  "55461196430",
  "24223731423",
  "55785767691",
  "94203145475",
  "24448720888",
  "32695806655",
  "26194491496",
  "40670113236",
  "23565064436",
  "26753580946",
  "52104650064",
  "25916816944",
  "23658969402",
  "93417571520",
  "72119178685",
  "64825448365",
  "67608421622",
  "78296503861",
  "12776677840",
  "14379808989",
  "35007365989",
  "73937415673",
  "63470361707",
  "94385446506",
  "80796089602",
  "99650263600",
  "16115785696",
  "88926992002",
  "05971243002",
  "80724798811",
  "30730992731",
  "66824905168",
  "32709805910",
  "36391974337",
  "51687024272",
  "42217278750",
  "46835139169",
  "87966920026",
  "68615290288",
  "17158508900",
  "04122515175",
  "47776618252",
  "81163599058",
  "72555251840",
  "40252913369",
  "17302175006",
  "65802463460",
  "95236739515",
  "77831112111",
  "60895258850",
  "32686162352",
  "83429985530",
  "77656992681",
  "08748724320",
  "76134894364",
  "59088867502",
  "75709395603",
  "32925844963",
  "69968719390",
  "18646585698",
  "24447289004",
  "60708296809",
  "15391191851",
  "52095072662",
  "87700000787",
  "95359180031",
  "22514105210",
  "27873138187",
  "40588624765",
  "01401149860",
  "24611466509",
  "25382690677",
  "32109029105",
  "26981650016",
  "75734570881",
  "33580372940",
  "91465886701",
  "31888095733",
  "17707627061",
  "40281722972",
  "98970828843",
  "49300791769",
  "25292524058",
  "89924110262",
  "12135136787",
  "10357291330",
  "02197840904",
  "34217543537",
  "22772829484",
  "55926143144",
  "69966560847",
  "29374344298",
  "00087058484",
  "36322188308",
  "30890864520",
  "42301756230",
  "30988055935",
  "04234864987",
  "95175902700",
  "65069206858",
  "82075158919",
  "60685567670",
  "71443058921",
  "32365162302",
  "43293679741",
  "78504985767",
  "78623926417",
  "56689633036",
  "12397336037",
  "15321797686",
  "26725700159",
  "94959124381",
  "93854161819",
  "86702226290",
  "56829803572",
  "24431365103",
  "50370410724",
  "40676203004",
  "49343556400",
  "70706019310",
  "20623146938",
  "91295152306",
  "59493245799",
  "87387245976",
  "48053281364",
  "90529162894",
  "56550645946",
  "69047140805",
  "86678771754",
  "73453773286",
  "64902951478",
  "06042200026",
  "79773406273",
  "21570323172",
  "47123269504",
  "55277456545",
  "58409434729",
  "69511529302",
  "49863094084",
  "07316909243",
  "29490408170",
  "75295834255",
  "64319802840",
  "28201975702",
  "25132319601",
  "52874257627",
  "79143719870",
  "95924986130",
  "24553786078",
  "00789508025",
  "67150447901",
  "39978217938",
  "98328419461",
  "45638842600",
  "68753011850",
  "88398944652",
  "08985734697",
  "41593391900",
  "54770960908",
  "14986834228",
  "32724995870",
  "85372071434",
  "55942887006",
  "23835198781",
  "99947718363",
  "54595998831",
  "39412890810",
  "52463746828",
  "32961189619",
  "88299694778",
  "48758477178",
  "70543701024",
  "08000254518",
  "29371682015",
  "72022412805",
  "10202413360",
  "83162421638",
  "50383461770",
  "68265061603",
  "41897740102",
  "07567105002",
  "45352173257",
  "92782597458",
  "70947362764",
  "23844319650",
  "61895913770",
  "68467899857",
  "95023488835",
  "08555866959",
  "02758599136",
  "44884339110",
  "05014834502",
  "62640507383",
  "28761196177",
  "91952369641",
  "92356844002",
  "10903821720",
  "17493828184",
  "33443080215",
  "02846249983",
  "35936877034",
  "91929477846",
  "88291565263",
  "67968253587",
  "70351849601",
  "62832720120",
  "66914689042",
  "98157394776",
  "68399799584",
  "59927158974",
  "03730613910",
  "81040074785",
  "40075749955",
  "67300364040",
  "89023519918",
  "55790693703",
  "71837128855",
  "25089561701",
  "49167450080",
  "54696093823",
  "74576154646",
  "33463957528",
  "30456301191",
  "70342076410",
  "22276667773",
  "11736553670",
  "19822409171",
  "97534289404",
  "68590693884",
  "48808319428",
  "94092239250",
  "80946248491",
  "05087387411",
  "27678352537",
  "40184528878",
  "26795908596",
  "38042205225",
  "27050528258",
  "34276446649",
  "71631670490",
  "13399037009",
  "12354349175",
  "15589370782",
  "19805454800",
  "75896172307",
  "11961976140",
  "21281455466",
  "41121773802",
  "32483683340",
  "21034031090",
  "00702366578",
  "54966057590",
  "18781954471",
  "91058439150",
  "97263466174",
  "51974377506",
  "57519238339",
  "02777266378",
  "58132608191",
  "67256000412",
  "15152545727",
  "52657253432",
  "17479316971",
  "94440332707",
  "63664664725",
  "17206077239",
  "01305621645",
  "02291617311",
  "42996528325",
  "48237358780",
  "59121548900",
  "83011369238",
  "02336952960",
  "29531971313",
  "00414890800",
  "21356240721",
  "08449102592",
  "79764965841",
  "62630581110",
  "82415735180",
  "15864821701",
  "29195980381",
  "85751481736",
  "46023054578",
  "55550978146",
  "69064646988",
  "11986612136",
  "44699142311",
  "88979597393",
  "71068053030",
  "13848132275",
  "40162685372",
  "03411524319",
  "22628212559",
  "46437977937",
  "05683415534",
  "42630090072",
  "47356042247",
  "85205627749",
  "81416552300",
  "11294715706",
  "72929438868",
  "30144474575",
  "42797761408",
  "17423562851",
  "60317457731",
  "95387197836",
  "55818326962",
  "57986292729",
  "73707871618",
  "33570605852",
  "62069401150",
  "72550894022",
  "24561745118",
  "11480753438",
  "41139464138",
  "22434330552",
  "40841959807",
  "94946983102",
  "04476041701",
  "02445161514",
  "03767448349",
  "50503516206",
  "28561885605",
  "48636139780",
  "51765070742",
  "92606928292",
  "40097330800",
  "04039951166",
  "20280508503",
  "00419101303",
  "69650314560",
  "95527092808",
  "60580353383",
  "38647005155",
  "80240304093",
  "01916671337",
  "10624618050",
  "47280471722",
  "38507703416",
  "16083157328",
  "80721913841",
  "72596474128",
  "21189406661",
  "85026829121",
  "97567151997",
  "39821369003",
  "02078035404",
  "92860568787",
  "55862440948",
  "92942007326",
  "05081254700",
  "61220267511",
  "63994113413",
  "09027197432",
  "00763652663",
  "68270462801",
  "24793794777",
  "65939483242",
  "05326858232",
  "15701238150",
  "87718744956",
  "42376961368",
  "57881974990",
  "50776358997",
  "92572002907",
  "65502598924",
  "31963341761",
  "08450197460",
  "68484054727",
  "18348233381",
  "19403036576",
  "03480155963",
  "64390719230",
  "41824328902",
  "97639373289",
  "48419342107",
  "46280341844",
  "58682770822",
  "71985255319",
  "18888730010",
  "53173905146",
  "04221614101",
  "57490035155",
  "11744937601",
  "05229313116",
  "09371855321",
  "32988132801",
  "29511441566",
  "35114364501",
  "04140180690",
  "43793246782",
  "25301653305",
  "20357444960",
  "33531088400",
  "81031104825",
  "38828843840",
  "41249398285",
  "32443576759",
  "57343615334",
  "81347348024",
  "21562633691",
  "53997582052",
  "16331624954",
  "18502979725",
  "41699689428",
  "05616948060",
  "06616504330",
  "23993425080",
  "40273288127",
  "65003904213",
  "63050958874",
  "51696046501",
  "41721177725",
  "57177118006",
  "97984897005",
  "08343735722",
  "64790948224",
  "36286415107",
  "58673478412",
  "25153254647",
  "84383596711",
  "80746379722",
  "84351213123",
  "11750951285",
  "05607328085",
  "04009268102",
  "32030337633",
  "89957627085",
  "46295503504",
  "59777595704",
  "86352270679",
  "04327560179",
  "71759157201",
  "97708318599",
  "01878732008",
  "02916962206",
  "56595357172",
  "26814123430",
  "36714437800",
  "12431059215",
  "37456319060",
  "85858606000",
  "41699204252",
  "93865810375",
  "51986839114",
  "88871033458",
  "85785391992",
  "60492935692",
  "75741941557",
  "41343774290",
  "45383272721",
  "38599576257",
  "45053487540",
  "25795044080",
  "03175512702",
  "26871774283",
];
function generateCNH() {
  return cnhs[getRandomInt(0, cnhs.length - 1)];
}
function getRandomInt(min, max) {
  return Math.floor(Math.random() * (max - min + 1)) + min;
}
function generateName() {
  const names = [
    "Liam",
    "Emma",
    "Noah",
    "Olivia",
    "William",
    "Ava",
    "James",
    "Isabella",
    "Logan",
    "Sophia",
    "Benjamin",
    "Mia",
    "Mason",
    "Charlotte",
    "Elijah",
    "Amelia",
    "Oliver",
    "Evelyn",
    "Jacob",
    "Abigail",
    "Lucas",
    "Harper",
    "Michael",
    "Emily",
    "Alexander",
    "Elizabeth",
    "Ethan",
    "Avery",
    "Daniel",
    "Sofia",
    "Matthew",
    "Ella",
    "Aiden",
    "Madison",
    "Henry",
    "Scarlett",
    "Joseph",
    "Victoria",
    "Jackson",
    "Aria",
    "Samuel",
    "Grace",
    "Sebastian",
    "Chloe",
    "David",
    "Camila",
    "Carter",
    "Penelope",
    "Wyatt",
    "Riley",
    "Jayden",
    "Layla",
    "John",
    "Lillian",
    "Owen",
    "Nora",
    "Dylan",
    "Zoey",
    "Luke",
    "Mila",
    "Gabriel",
    "Aubrey",
    "Anthony",
    "Hannah",
    "Isaac",
    "Lily",
    "Grayson",
    "Addison",
    "Jack",
    "Eleanor",
    "Julian",
    "Natalie",
    "Levi",
    "Luna",
    "Christopher",
    "Savannah",
    "Joshua",
    "Brooklyn",
    "Andrew",
    "Leah",
    "Lincoln",
    "Zoe",
    "Mateo",
    "Stella",
    "Ryan",
    "Hazel",
    "Jaxon",
    "Ellie",
    "Nathan",
    "Paisley",
    "Aaron",
    "Audrey",
    "Isaiah",
    "Skylar",
    "Thomas",
    "Violet",
    "Charles",
    "Claire",
    "Caleb",
    "Bella",
    "Josiah",
    "Aurora",
    "Christian",
    "Lucy",
    "Hunter",
    "Anna",
    "Eli",
    "Samantha",
    "Jonathan",
    "Caroline",
    "Connor",
    "Genesis",
    "Landon",
    "Aaliyah",
    "Adrian",
    "Kennedy",
    "Asher",
    "Kinsley",
  ];
  return names[getRandomInt(0, names.length - 1)];
}
const pispasep = [
  "90067946947",
  "22139644874",
  "17502222764",
  "60275317432",
  "89734637863",
  "35419938381",
  "47304148772",
  "86586594865",
  "78669322322",
  "58798566444",
  "83387336659",
  "95643325103",
  "84875224653",
  "18564268413",
  "48687761756",
  "20848534063",
  "20551024903",
  "10886605510",
  "16905766110",
  "14873037189",
  "58303370167",
  "34257998327",
  "78365674237",
  "31060173815",
  "54937800359",
  "81881951958",
  "43695349868",
  "45211064297",
  "20064111592",
  "30706977035",
  "06895789204",
  "24835055326",
  "70958329258",
  "15151377756",
  "29597335298",
  "32093935690",
  "89984064003",
  "18161144449",
  "56999212198",
  "98213483875",
  "21353345051",
  "48426283478",
  "73247558139",
  "34770999560",
  "48202076266",
  "69145832446",
  "14536469474",
  "59858170159",
  "79151291025",
  "81737369376",
  "38593529445",
  "62641478623",
  "49837860876",
  "58550428382",
  "92233752252",
  "41811665205",
  "11957013200",
  "17524497472",
  "11093641490",
  "29314089770",
  "76366628330",
  "85190508840",
  "56855065244",
  "10447792854",
  "84605498653",
  "22590258142",
  "70197172203",
  "28011563958",
  "90417278883",
  "46274531753",
  "25396179733",
  "53296773396",
  "75997962579",
  "72683316297",
  "87406404092",
  "96486877927",
  "57609538733",
  "91207537929",
  "89047785863",
  "87727378100",
  "62836682742",
  "64338740640",
  "32602105798",
  "25939513582",
  "48718693706",
  "85086429735",
  "44046557506",
  "81006034710",
  "59130986836",
  "79345352313",
  "12500066446",
  "34219648381",
  "09553927470",
  "74107684156",
  "06887769313",
  "08143651019",
  "88408949469",
  "20873005389",
  "56679692253",
  "75142632211",
  "03655035758",
  "73542807993",
  "46808469019",
  "67065130872",
  "26841822399",
  "85866944566",
  "07853639857",
  "16733064520",
  "34899653469",
  "53698787286",
  "87874291947",
  "22547172770",
  "91447983907",
  "55975642640",
  "25489403552",
  "33900094458",
  "15362568489",
  "29388767446",
  "56279889998",
  "37395781441",
  "67237954754",
  "76817558225",
  "57510522207",
  "33254056322",
  "95744224345",
  "66804986226",
  "72481665029",
  "02146210278",
  "54308851527",
  "06418873800",
  "27521149489",
  "59056373155",
  "90246843662",
  "25744762478",
  "96858716814",
  "51843598829",
  "56189643027",
  "53405189770",
  "31563041188",
  "06728180170",
  "48514249336",
  "60089961203",
  "60041584256",
  "44819127819",
  "89255048590",
  "13481244044",
  "41995925574",
  "90892051260",
  "28553507273",
  "58689277913",
  "96725944620",
  "54904070475",
  "94451720960",
  "92266909665",
  "71075433833",
  "03114585002",
  "33439515552",
  "86245218421",
  "92329599757",
  "24085463806",
  "38066998945",
  "64264632466",
  "67017704643",
  "14850942683",
  "37847353272",
  "01949909333",
  "38444394408",
  "16503513314",
  "12370609526",
  "40276722816",
  "84986250270",
  "41411819230",
  "44301104397",
  "75381870830",
  "48873180768",
  "07008997516",
  "50938651168",
  "65188737833",
  "92212492405",
  "41887461759",
  "45299398439",
  "34221277247",
  "35603919610",
  "66615197607",
  "22815181818",
  "55881074206",
  "66683102063",
  "09799486136",
  "61268600904",
  "98074505794",
  "01944333840",
  "75032419254",
  "63493828492",
  "58528749511",
  "69103375934",
  "62986236764",
  "06756141523",
  "79131400830",
  "67669905012",
  "25565092281",
  "08989002883",
  "66286635154",
  "70997797534",
  "12039826507",
  "89874868833",
  "90434270569",
  "35952550850",
  "46923605728",
  "21886098036",
  "35529421740",
  "35490751450",
  "54555153672",
  "68070478120",
  "76611132982",
  "62888138970",
  "97681761132",
  "55556396216",
  "13772851770",
  "17481539462",
  "46858378572",
  "92022267761",
  "46670724392",
  "04898178633",
  "26207066160",
  "54086825827",
  "35908590540",
  "79237296882",
  "18012623540",
  "22390237249",
  "58599734444",
  "36079475850",
  "56465570173",
  "37433841850",
  "88212634600",
  "08333398047",
  "16565896160",
  "19837280436",
  "27360303856",
  "28677038798",
  "62618505792",
  "30794391313",
  "10815044051",
  "87496683794",
  "18347494202",
  "55039699916",
  "26833824205",
  "22029248923",
  "53425646255",
  "43777153193",
  "24189380362",
  "62226712877",
  "57948750700",
  "26931930140",
  "85653444140",
  "16809610180",
  "10097436400",
  "40884082503",
  "78101424104",
  "57057803980",
  "90060653723",
  "57306095479",
  "67950981886",
  "08185742136",
  "55254578729",
  "58288391995",
  "49250189197",
  "86803961135",
  "41936167561",
  "09314459034",
  "54279183525",
  "80752238068",
  "79435887364",
  "91179742926",
  "84998500250",
  "94624310866",
  "42003846186",
  "59836171411",
  "88298502821",
  "91163968172",
  "99475034405",
  "79069235230",
  "83857002425",
  "41550928534",
  "69356118980",
  "03125046620",
  "92016033312",
  "57370910641",
  "31867272732",
  "15078374268",
  "75836939001",
  "37561329914",
  "61018032640",
  "52360956010",
  "59562745572",
  "66134643545",
  "15540973317",
  "02371541464",
  "76614886341",
  "28252282957",
  "11425174889",
  "05170920330",
  "70602132291",
  "14655109161",
  "39243998410",
  "81585565402",
  "41034096245",
  "95408589341",
  "19610849434",
  "23729634751",
  "86522743320",
  "71777151285",
  "27893347576",
  "31347709834",
  "50042998995",
  "69904402140",
  "13657179126",
  "56696204601",
  "67650984391",
  "98135582939",
  "16493758689",
  "92414693088",
  "30304940510",
  "80065007164",
  "28545780985",
  "67007675527",
  "22027797865",
  "17066793666",
  "61159225657",
  "12295003794",
  "99562076360",
  "12228707335",
  "54823617385",
  "29514289110",
  "18880798400",
  "83539731970",
  "34753483194",
  "52774586900",
  "83326228627",
  "07844974967",
  "87988368760",
  "11813637410",
  "11355744746",
  "73003080086",
  "63927296746",
  "61610909799",
  "43773027487",
  "66546840828",
  "82136651121",
  "46882055885",
  "67916834060",
  "88270972300",
  "91885872090",
  "68515686831",
  "88512887032",
  "44135981567",
  "47214203367",
  "10562336351",
  "07945248854",
  "67515516814",
  "59848822501",
  "11863189445",
  "82061565908",
  "63064058300",
  "14143736094",
  "14808046780",
  "32736331920",
  "51281818147",
  "98457641275",
  "85416276457",
  "44037549830",
  "80421545540",
  "98975410963",
  "35329498013",
  "26624082460",
  "81433628940",
  "66322669599",
  "18847310852",
  "31775640476",
  "04448432938",
  "61047097720",
  "09468012510",
  "29212627277",
  "22403393915",
  "26629818413",
  "74655200405",
  "82306446671",
  "67351596268",
  "30258263328",
  "89009610270",
  "72282637793",
  "70517253586",
  "55021641886",
  "59535767218",
  "77385496760",
];
function generatePISPASEP() {
  return pispasep[getRandomInt(0, pispasep.length - 1)];
}

const creditcard = [
  {
    number: "4929 5316 6633 2725 ",
    date: "07/02/2025",
    code: "357",
  },
  {
    number: "4929 4975 9461 1772 ",
    date: "07/04/2025",
    code: "800",
  },
  {
    number: "4485 1057 0242 8817 ",
    date: "07/11/2024",
    code: "582",
  },
  {
    number: "4916 2181 3755 2090 ",
    date: "07/07/2024",
    code: "255",
  },
  {
    number: "4929 3757 1212 2940 ",
    date: "07/04/2025",
    code: "346",
  },
  {
    number: "4024 0071 7982 9030 ",
    date: "07/06/2024",
    code: "891",
  },
  {
    number: "4716 0410 6261 6553 ",
    date: "07/08/2024",
    code: "732",
  },
  {
    number: "4539 0584 6655 8962 ",
    date: "07/11/2025",
    code: "191",
  },
  {
    number: "4485 4479 2036 9633 ",
    date: "07/08/2024",
    code: "990",
  },
  {
    number: "4485 9690 3693 2095 ",
    date: "07/04/2024",
    code: "596",
  },
  {
    number: "4716 3357 3766 1428 ",
    date: "07/07/2025",
    code: "338",
  },
  {
    number: "4929 4485 4838 7969 ",
    date: "07/11/2024",
    code: "933",
  },
  {
    number: "4716 3217 8872 8308 ",
    date: "07/10/2024",
    code: "770",
  },
  {
    number: "4539 4556 6733 5283 ",
    date: "07/08/2025",
    code: "408",
  },
  {
    number: "4716 7464 7333 7843 ",
    date: "07/11/2025",
    code: "807",
  },
  {
    number: "4916 3701 0761 5041 ",
    date: "07/05/2025",
    code: "650",
  },
  {
    number: "4532 4295 1406 0342 ",
    date: "07/05/2025",
    code: "541",
  },
  {
    number: "4024 0071 2352 6518 ",
    date: "07/08/2025",
    code: "522",
  },
  {
    number: "4916 2940 2560 4913 ",
    date: "07/03/2025",
    code: "450",
  },
  {
    number: "4485 1395 1992 5590 ",
    date: "07/08/2024",
    code: "397",
  },
  {
    number: "4532 0176 1379 2913 ",
    date: "07/12/2024",
    code: "890",
  },
  {
    number: "4929 1602 0117 5082 ",
    date: "07/01/2025",
    code: "802",
  },
  {
    number: "4024 0071 5314 9942 ",
    date: "07/01/2025",
    code: "445",
  },
  {
    number: "4556 2600 4002 0141 ",
    date: "07/02/2025",
    code: "465",
  },
  {
    number: "4532 3285 9064 6331 ",
    date: "07/06/2025",
    code: "622",
  },
  {
    number: "4485 4022 3064 4035 ",
    date: "07/04/2025",
    code: "107",
  },
  {
    number: "4024 0071 7580 0076 ",
    date: "07/02/2025",
    code: "709",
  },
  {
    number: "4929 7955 9445 7173 ",
    date: "07/03/2025",
    code: "829",
  },
  {
    number: "4024 0071 1445 2559 ",
    date: "07/08/2025",
    code: "410",
  },
  {
    number: "4024 0071 9904 4339 ",
    date: "07/10/2024",
    code: "712",
  },
  {
    number: "4024 0071 4009 9077 ",
    date: "07/08/2024",
    code: "696",
  },
  {
    number: "4532 2461 4134 6906 ",
    date: "07/06/2024",
    code: "520",
  },
  {
    number: "4556 0876 2368 7941 ",
    date: "07/11/2024",
    code: "662",
  },
  {
    number: "4916 9443 8115 1830 ",
    date: "07/05/2025",
    code: "398",
  },
  {
    number: "4539 2840 5731 3993 ",
    date: "07/04/2025",
    code: "283",
  },
  {
    number: "4556 3594 0986 7132 ",
    date: "07/05/2024",
    code: "887",
  },
  {
    number: "4869 0116 8305 9991 ",
    date: "07/08/2024",
    code: "585",
  },
  {
    number: "4485 4877 3166 3725 ",
    date: "07/07/2024",
    code: "393",
  },
  {
    number: "4916 5317 8043 8510 ",
    date: "07/02/2025",
    code: "732",
  },
  {
    number: "4916 9490 7948 3288 ",
    date: "07/09/2024",
    code: "167",
  },
  {
    number: "4024 0071 3556 1933 ",
    date: "07/06/2024",
    code: "978",
  },
  {
    number: "4532 8505 2293 4733 ",
    date: "07/11/2025",
    code: "360",
  },
  {
    number: "4532 6152 4252 3722 ",
    date: "07/10/2025",
    code: "554",
  },
  {
    number: "4532 2856 1927 6542 ",
    date: "07/03/2025",
    code: "627",
  },
  {
    number: "4929 9335 2018 5838 ",
    date: "07/11/2024",
    code: "374",
  },
  {
    number: "4916 2460 8812 1961 ",
    date: "07/06/2024",
    code: "586",
  },
  {
    number: "4916 1038 8564 3970 ",
    date: "07/11/2025",
    code: "684",
  },
  {
    number: "4556 2980 0034 1248 ",
    date: "07/08/2024",
    code: "846",
  },
  {
    number: "4556 2845 7033 0563 ",
    date: "07/08/2024",
    code: "310",
  },
  {
    number: "4716 2808 3355 0424 ",
    date: "07/06/2025",
    code: "712",
  },
  {
    number: "4024 0071 3953 4449 ",
    date: "07/05/2025",
    code: "472",
  },
  {
    number: "4368 4183 3842 8735 ",
    date: "07/09/2024",
    code: "836",
  },
  {
    number: "4485 0520 9274 0528 ",
    date: "07/07/2024",
    code: "800",
  },
  {
    number: "4539 2818 6161 7841 ",
    date: "07/09/2024",
    code: "403",
  },
  {
    number: "4556 7410 5196 1950 ",
    date: "07/03/2025",
    code: "295",
  },
  {
    number: "4024 0071 5352 7345 ",
    date: "07/04/2024",
    code: "114",
  },
  {
    number: "4024 0071 3355 0458 ",
    date: "07/06/2025",
    code: "551",
  },
  {
    number: "4532 2679 4024 9380 ",
    date: "07/09/2024",
    code: "579",
  },
  {
    number: "4929 9411 4292 1035 ",
    date: "07/07/2024",
    code: "764",
  },
  {
    number: "4654 2658 4030 4497 ",
    date: "07/08/2025",
    code: "239",
  },
  {
    number: "4532 8598 1547 9918 ",
    date: "07/01/2025",
    code: "550",
  },
  {
    number: "4716 5523 2996 4909 ",
    date: "07/07/2025",
    code: "350",
  },
  {
    number: "4916 0200 7118 5204 ",
    date: "07/03/2025",
    code: "138",
  },
  {
    number: "4724 8433 6115 0246 ",
    date: "07/06/2024",
    code: "209",
  },
  {
    number: "4024 0071 0297 9621 ",
    date: "07/07/2025",
    code: "604",
  },
  {
    number: "4556 8690 6207 9379 ",
    date: "07/11/2024",
    code: "305",
  },
  {
    number: "4485 3727 1941 2016 ",
    date: "07/08/2024",
    code: "222",
  },
  {
    number: "4532 1235 3687 1386 ",
    date: "07/04/2024",
    code: "661",
  },
  {
    number: "4929 5010 7146 9370 ",
    date: "07/05/2025",
    code: "560",
  },
  {
    number: "4556 4373 5623 0502 ",
    date: "07/03/2025",
    code: "393",
  },
  {
    number: "4929 1953 4727 9638 ",
    date: "07/04/2024",
    code: "664",
  },
  {
    number: "4532 6395 3140 8346 ",
    date: "07/05/2024",
    code: "900",
  },
  {
    number: "4556 6785 7416 1918 ",
    date: "07/05/2025",
    code: "739",
  },
  {
    number: "4716 0277 9918 3157 ",
    date: "07/05/2025",
    code: "457",
  },
  {
    number: "4024 0071 3045 2666 ",
    date: "07/09/2025",
    code: "434",
  },
  {
    number: "4916 9745 7723 8745 ",
    date: "07/04/2025",
    code: "766",
  },
  {
    number: "4716 0676 8577 3364 ",
    date: "07/02/2025",
    code: "480",
  },
  {
    number: "4716 8257 4977 2940 ",
    date: "07/07/2024",
    code: "292",
  },
  {
    number: "4024 0071 5040 5966 ",
    date: "07/09/2024",
    code: "222",
  },
  {
    number: "4485 0609 7317 0970 ",
    date: "07/07/2025",
    code: "828",
  },
  {
    number: "4929 2116 0857 1629 ",
    date: "07/03/2025",
    code: "489",
  },
  {
    number: "4024 0071 1990 8357 ",
    date: "07/06/2025",
    code: "117",
  },
  {
    number: "4214 5796 9018 6579 ",
    date: "07/03/2025",
    code: "225",
  },
  {
    number: "4716 7730 3818 5764 ",
    date: "07/04/2025",
    code: "672",
  },
  {
    number: "4916 2059 1055 3643 ",
    date: "07/01/2025",
    code: "352",
  },
  {
    number: "4916 6595 2076 4174 ",
    date: "07/08/2025",
    code: "923",
  },
  {
    number: "4485 5341 6384 4842 ",
    date: "07/07/2025",
    code: "125",
  },
  {
    number: "4532 5218 8846 8596 ",
    date: "07/08/2024",
    code: "354",
  },
  {
    number: "4539 4233 9122 3043 ",
    date: "07/02/2025",
    code: "116",
  },
  {
    number: "4929 7678 9971 6058 ",
    date: "07/09/2024",
    code: "635",
  },
  {
    number: "4018 9028 2509 0144 ",
    date: "07/06/2024",
    code: "704",
  },
  {
    number: "4929 9460 9335 3872 ",
    date: "07/11/2025",
    code: "603",
  },
  {
    number: "4716 6643 8871 0089 ",
    date: "07/07/2024",
    code: "962",
  },
  {
    number: "4716 7111 9751 9027 ",
    date: "07/11/2024",
    code: "981",
  },
  {
    number: "4916 1788 8872 0454 ",
    date: "07/04/2025",
    code: "956",
  },
  {
    number: "4485 4121 8021 5546 ",
    date: "07/01/2025",
    code: "696",
  },
  {
    number: "4929 8007 0039 5927 ",
    date: "07/07/2024",
    code: "693",
  },
  {
    number: "4556 1476 3058 4950 ",
    date: "07/03/2025",
    code: "193",
  },
  {
    number: "4087 4457 3803 4805 ",
    date: "07/03/2025",
    code: "159",
  },
  {
    number: "4716 1303 3811 9672 ",
    date: "07/06/2025",
    code: "983",
  },
  {
    number: "4024 0071 2637 5327 ",
    date: "07/06/2025",
    code: "756",
  },
  {
    number: "4929 9860 1962 9856 ",
    date: "07/11/2024",
    code: "295",
  },
  {
    number: "4539 7613 9136 6621 ",
    date: "07/07/2024",
    code: "341",
  },
  {
    number: "4532 8804 2258 7000 ",
    date: "07/12/2024",
    code: "949",
  },
  {
    number: "4024 0071 6616 0894 ",
    date: "07/04/2025",
    code: "480",
  },
  {
    number: "4916 7368 5174 3009 ",
    date: "07/10/2024",
    code: "447",
  },
  {
    number: "4532 5843 7177 5430 ",
    date: "07/10/2024",
    code: "822",
  },
  {
    number: "4716 2850 9984 2852 ",
    date: "07/07/2024",
    code: "977",
  },
  {
    number: "4556 1957 9625 0619 ",
    date: "07/11/2025",
    code: "873",
  },
  {
    number: "4716 3574 4171 3998 ",
    date: "07/06/2024",
    code: "696",
  },
  {
    number: "4485 1038 0619 5309 ",
    date: "07/06/2025",
    code: "879",
  },
  {
    number: "4716 2556 2560 7442 ",
    date: "07/06/2025",
    code: "751",
  },
  {
    number: "4539 0699 9155 9367 ",
    date: "07/04/2024",
    code: "314",
  },
  {
    number: "4532 2621 3736 3292 ",
    date: "07/06/2024",
    code: "841",
  },
  {
    number: "4716 2448 1540 8118 ",
    date: "07/10/2025",
    code: "629",
  },
  {
    number: "4916 2182 0425 2228 ",
    date: "07/01/2025",
    code: "114",
  },
  {
    number: "4916 6998 1829 6117 ",
    date: "07/06/2025",
    code: "742",
  },
  {
    number: "4929 0857 7679 3615 ",
    date: "07/03/2025",
    code: "327",
  },
  {
    number: "4916 7321 7852 8793 ",
    date: "07/11/2024",
    code: "787",
  },
  {
    number: "4024 0071 4168 6765 ",
    date: "07/02/2025",
    code: "656",
  },
  {
    number: "4485 0099 6182 0088 ",
    date: "07/08/2024",
    code: "184",
  },
  {
    number: "4024 0071 2585 7556 ",
    date: "07/06/2025",
    code: "907",
  },
  {
    number: "4539 4402 6391 3824 ",
    date: "07/06/2025",
    code: "709",
  },
  {
    number: "4916 5189 4466 8124 ",
    date: "07/03/2025",
    code: "620",
  },
  {
    number: "4929 2117 4325 7522 ",
    date: "07/10/2025",
    code: "854",
  },
  {
    number: "4929 0137 7491 8455 ",
    date: "07/05/2024",
    code: "588",
  },
  {
    number: "4556 7929 9314 5426 ",
    date: "07/07/2025",
    code: "791",
  },
  {
    number: "4916 2598 0059 1809 ",
    date: "07/08/2025",
    code: "714",
  },
  {
    number: "4929 7293 0889 8893 ",
    date: "07/03/2025",
    code: "680",
  },
  {
    number: "4929 3237 2202 3328 ",
    date: "07/06/2024",
    code: "982",
  },
  {
    number: "4024 0071 3529 4253 ",
    date: "07/05/2025",
    code: "563",
  },
  {
    number: "4485 5002 6841 9731 ",
    date: "07/09/2024",
    code: "696",
  },
  {
    number: "4532 2559 8474 6023 ",
    date: "07/06/2024",
    code: "221",
  },
  {
    number: "4532 6340 0063 5727 ",
    date: "07/11/2025",
    code: "758",
  },
  {
    number: "4485 4758 7224 1278 ",
    date: "07/04/2024",
    code: "416",
  },
  {
    number: "4532 6951 6451 5609 ",
    date: "07/11/2025",
    code: "917",
  },
  {
    number: "4716 6493 9502 0010 ",
    date: "07/10/2025",
    code: "621",
  },
  {
    number: "4929 1942 5566 9930 ",
    date: "07/12/2024",
    code: "703",
  },
  {
    number: "4485 6097 5930 8772 ",
    date: "07/09/2025",
    code: "968",
  },
  {
    number: "4929 8370 5064 2185 ",
    date: "07/08/2024",
    code: "298",
  },
  {
    number: "4539 8174 5101 6645 ",
    date: "07/06/2025",
    code: "432",
  },
  {
    number: "4929 8688 7400 7487 ",
    date: "07/07/2025",
    code: "556",
  },
  {
    number: "4539 3866 6733 1297 ",
    date: "07/10/2024",
    code: "613",
  },
  {
    number: "4539 9049 8535 1183 ",
    date: "07/04/2024",
    code: "280",
  },
  {
    number: "4929 2161 3957 3671 ",
    date: "07/05/2025",
    code: "768",
  },
  {
    number: "4716 1699 6652 0629 ",
    date: "07/05/2025",
    code: "834",
  },
  {
    number: "4405 7353 4219 2972 ",
    date: "07/11/2025",
    code: "121",
  },
  {
    number: "4024 0071 8963 2440 ",
    date: "07/10/2025",
    code: "966",
  },
  {
    number: "4916 7639 2997 4002 ",
    date: "07/10/2024",
    code: "122",
  },
  {
    number: "4539 7439 4908 0474 ",
    date: "07/09/2025",
    code: "727",
  },
];
function generateCreditCard() {
  const randomIndex = getRandomInt(0, creditcard.length - 1);
  const card = { ...creditcard[randomIndex] }; // Cria uma cópia do objeto
  const currentYear = new Date().getFullYear();
  const futureYear = currentYear + 2;

  const dateParts = card.date.split("/");
  const month = dateParts[0];
  const year = futureYear.toString();
  card.date = `${month}/${year}`;

  return card;
}

function generateAge(min = 18, max = 60) {
  return getRandomInt(min, max);
}

function generateRNE() {
  const letter = "W";

  const randomNumber = Math.floor(Math.random() * 100000000)
    .toString()
    .padStart(8, "0");

  const checkDigit = Math.floor(Math.random() * 10);

  return `${letter}${randomNumber}-${checkDigit}`;
}
function generatePlate() {
  const randomLetter = () =>
    String.fromCharCode(65 + Math.floor(Math.random() * 26));

  const randomDigit = () => Math.floor(Math.random() * 10);

  const oldPlate = `${randomLetter()}${randomLetter()}${randomLetter()}-${randomDigit()}${randomDigit()}${randomDigit()}${randomDigit()}`;

  const mercosulPlate = `${randomLetter()}${randomLetter()}${randomLetter()}${randomDigit()}${randomLetter()}${randomDigit()}${randomDigit()}`;

  return {
    oldFormat: oldPlate,
    mercosulFormat: mercosulPlate,
  };
}

function generateUuidv4() {
  return v4();
}

function generateLoremIpsum(wordCount = 1) {
  const loremIpsum =
    "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.";

  const words = loremIpsum.split(" ");
  const totalWords = words.length;
  let result = "";

  for (let i = 0; i < wordCount; i++) {
    result += words[i % totalWords] + " ";
  }

  result = result.trim() + ".";

  return result;
}

function generateDate(day = 0, month = 0, year = 0) {
  const resultDate = new Date();

  resultDate.setDate(resultDate.getDate() + day);
  resultDate.setMonth(resultDate.getMonth() + month);
  resultDate.setFullYear(resultDate.getFullYear() + year);

  function padTo2Digits(num) {
    return num.toString().padStart(2, "0");
  }

  function brazilianFormat(date) {
    return {
      date: `${padTo2Digits(date.getDate())}/${padTo2Digits(
        date.getMonth() + 1
      )}/${date.getFullYear()}`,
      time: `${padTo2Digits(date.getHours())}:${padTo2Digits(
        date.getMinutes()
      )}:${padTo2Digits(date.getSeconds())}`,
      dateTime: `${padTo2Digits(date.getDate())}/${padTo2Digits(
        date.getMonth() + 1
      )}/${date.getFullYear()} ${padTo2Digits(date.getHours())}:${padTo2Digits(
        date.getMinutes()
      )}:${padTo2Digits(date.getSeconds())}`,
      timeStamp: date.getTime(),
      iso: date.toISOString(),
      stringFormat: date.toString(),
    };
  }

  const dateFormats = {
    iso: resultDate.toISOString(),
    utc: resultDate.toUTCString(),
    stringFormat: resultDate.toString(),
    localDate: resultDate.toLocaleDateString(),
    localTime: resultDate.toLocaleTimeString(),
    localDateTime: resultDate.toLocaleString(),
    timestamp: resultDate.getTime(),
    customFormat: `${resultDate.getFullYear()}-${padTo2Digits(
      resultDate.getMonth() + 1
    )}-${padTo2Digits(resultDate.getDate())} ${padTo2Digits(
      resultDate.getHours()
    )}:${padTo2Digits(resultDate.getMinutes())}:${padTo2Digits(
      resultDate.getSeconds()
    )}`,
    br: brazilianFormat(resultDate),
  };

  return dateFormats;
}
function generatePassword(length = 10) {
  const charset =
    "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789!@#$%^&*()_+~`|}{[]:;?><,./-=";
  let password = "";
  for (let i = 0, n = charset.length; i < length; ++i) {
    password += charset.charAt(Math.floor(Math.random() * n));
  }
  return password;
}

function generateFakePolicyNumber(length = 10) {
  const characters = "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789";
  let policyNumber = "";

  for (let i = 0; i < length; i++) {
    const randomIndex = Math.floor(Math.random() * characters.length);
    policyNumber += characters.charAt(randomIndex);
  }

  return policyNumber;
}
const cars = [
  {
    marca: "Hyundai",
    modelo: "Elantra GLS 1.6",
    renavam: "20938726328",
    placa: "IDG-0919",
    cor: "Dourado",
    ano: "1992",
  },
  {
    marca: "LAMBORGHINI",
    modelo: "Gallardo Coupe LP550-2 Bicolore",
    renavam: "61628445905",
    placa: "KBW-7815",
    cor: "Vermelho",
    ano: "2012",
  },
  {
    marca: "LIFAN",
    modelo: "FOISON 1.3 16V 85cv 2p",
    renavam: "08105980647",
    placa: "HWF-6534",
    cor: "Laranja",
    ano: "2014",
  },
  {
    marca: "Volvo",
    modelo: "S60 T-5 MOMENTUM 2.0 245cv 4p",
    renavam: "81036771550",
    placa: "MZC-7196",
    cor: "Branco",
    ano: "2015",
  },
  {
    marca: "TAC",
    modelo: "Stark 2.3 4WD 127cv TDI Diesel 3p",
    renavam: "01926747221",
    placa: "HWA-7418",
    cor: "Marrom",
    ano: "2010",
  },
  {
    marca: "Mazda",
    modelo: "RX 7 2.6 Turbo",
    renavam: "44593524170",
    placa: "MWR-7141",
    cor: "Laranja",
    ano: "1993",
  },
  {
    marca: "Troller",
    modelo: "T-4 4x4 3.2 TGV tdi Cap. R",
    renavam: "80134653627",
    placa: "JFS-5772",
    cor: "Prata",
    ano: "2013",
  },
  {
    marca: "JAC",
    modelo: "J3 turin Sedan Brasil 1.4 16V 4p Mec.",
    renavam: "22513182137",
    placa: "NBC-9372",
    cor: "Dourado",
    ano: "2012",
  },
  {
    marca: "Saturn",
    modelo: "SL-2 1.9",
    renavam: "78835881578",
    placa: "KBX-9083",
    cor: "Cinza",
    ano: "1991",
  },
  {
    marca: "Hyundai",
    modelo: "Atos Prime GLS 1.0 Semi-Aut.",
    renavam: "26176525762",
    placa: "MUU-0880",
    cor: "Vermelho",
    ano: "2000",
  },
  {
    marca: "Asia Motors",
    modelo: "AM-825 Luxo 4.0 Diesel",
    renavam: "97127926391",
    placa: "LCA-9654",
    cor: "Prata",
    ano: "1994",
  },
  {
    marca: "Porsche",
    modelo: "911 Carrera Evo Cabriolet-4",
    renavam: "36839539658",
    placa: "HCZ-7761",
    cor: "Prata",
    ano: "1998",
  },
  {
    marca: "BRM",
    modelo: "Buggy W. Turing/Luxo/M-11 Absolvt 1.6 8V",
    renavam: "32712366855",
    placa: "JVK-4428",
    cor: "Branco",
    ano: "2005",
  },
  {
    marca: "Maserati",
    modelo: "Gran Turismo Sport 4.7 V8 32v 460cv",
    renavam: "81361668224",
    placa: "NEM-7570",
    cor: "Verde",
    ano: "2013",
  },
  {
    marca: "MG",
    modelo: "MG6 1.8 16V Turbo 170cv Aut.",
    renavam: "32629457075",
    placa: "LWF-9287",
    cor: "Dourado",
    ano: "2011",
  },
  {
    marca: "VW - VolksWagen",
    modelo: "Gol Comfortline 1.0 T. Flex 8V 3p",
    renavam: "44898806927",
    placa: "JIT-2388",
    cor: "Dourado",
    ano: "2015",
  },
  {
    marca: "CHANA",
    modelo: "Utility 1.0 8V 53cv  (Furg",
    renavam: "40726211868",
    placa: "NBD-3590",
    cor: "Azul",
    ano: "2005",
  },
  {
    marca: "CHANGAN",
    modelo: "MINI STAR Utility 1.0 8V 53cv (Furg",
    renavam: "25225755489",
    placa: "NAK-8052",
    cor: "Dourado",
    ano: "2012",
  },
  {
    marca: "Fibravan",
    modelo: "Buggy Vip 1.8 8V",
    renavam: "93428991068",
    placa: "EPW-8069",
    cor: "Dourado",
    ano: "2006",
  },
  {
    marca: "Agrale",
    modelo: "MARRU",
    renavam: "62544015525",
    placa: "HVL-1017",
    cor: "Cinza",
    ano: "2004",
  },
  {
    marca: "ASTON MARTIN",
    modelo: "Rapide 6.0 V12 477cv",
    renavam: "26356055608",
    placa: "BBM-2603",
    cor: "Prata",
    ano: "2012",
  },
  {
    marca: "Gurgel",
    modelo: "Carajas Diesel",
    renavam: "28622985221",
    placa: "KKH-4393",
    cor: "Verde",
    ano: "1985",
  },
  {
    marca: "LIFAN",
    modelo: "620 TALENT 1.6 16V 106cv 4p",
    renavam: "15124889121",
    placa: "HWU-5477",
    cor: "Bege",
    ano: "2009",
  },
  {
    marca: "Nissan",
    modelo: "XTerra SE 4x4 2.8 132/140cv TB Int.Dies.",
    renavam: "95341034947",
    placa: "EML-6668",
    cor: "Prata",
    ano: "2003",
  },
  {
    marca: "Lada",
    modelo: "Niva 1.7i 4x4",
    renavam: "94440827556",
    placa: "MWC-5471",
    cor: "Vermelho",
    ano: "1998",
  },
  {
    marca: "VW - VolksWagen",
    modelo: "Saveiro 2.0 Mi",
    renavam: "24511681612",
    placa: "HPP-1825",
    cor: "Prata",
    ano: "2000",
  },
  {
    marca: "Fibravan",
    modelo: "Buggy Plus 1.6 8V",
    renavam: "81288658490",
    placa: "NBV-2391",
    cor: "Preto",
    ano: "1989",
  },
  {
    marca: "CBT Jipe",
    modelo: "Javali 3.0 4x4 Diesel",
    renavam: "59088303217",
    placa: "MYN-4812",
    cor: "Dourado",
    ano: "1988",
  },
  {
    marca: "SSANGYONG",
    modelo: "Rexton II 2.7 20V 165/186cv TDI Dies.Aut",
    renavam: "02049316466",
    placa: "NBM-5929",
    cor: "Preto",
    ano: "2006",
  },
  {
    marca: "Cadillac",
    modelo: "Deville/Eldorado 4.9",
    renavam: "82745476800",
    placa: "JYL-5705",
    cor: "Laranja",
    ano: "1991",
  },
  {
    marca: "Matra",
    modelo: "Pick-Up 4x2 Curto/Longo 2.5 TDI Diesel",
    renavam: "25108474246",
    placa: "HKN-8483",
    cor: "Bege",
    ano: "2002",
  },
  {
    marca: "Saab",
    modelo: "9000 CD 2.3 Turbo",
    renavam: "96814203676",
    placa: "NEZ-1409",
    cor: "Cinza",
    ano: "1992",
  },
  {
    marca: "Rover",
    modelo: "Mini Cooper 1.3",
    renavam: "11462420081",
    placa: "HPQ-5552",
    cor: "Branco",
    ano: "1992",
  },
  {
    marca: "Buggy",
    modelo: "Buggy 1.6/ TST/ RS 1.6 4-Lug.",
    renavam: "00753589524",
    placa: "KFP-7404",
    cor: "Prata",
    ano: "1985",
  },
  {
    marca: "Toyota",
    modelo: "Hilux CD SRV 4x2 3.0 8V 90cv Diesel",
    renavam: "55436945808",
    placa: "MOS-0196",
    cor: "Verde",
    ano: "2002",
  },
  {
    marca: "Hyundai",
    modelo: "Sonata GL 2.0 4p",
    renavam: "48217097264",
    placa: "KEQ-7143",
    cor: "Azul",
    ano: "1992",
  },
  {
    marca: "Peugeot",
    modelo: "605 SRi/ SLi 2.0",
    renavam: "02464692152",
    placa: "AXP-8138",
    cor: "Amarelo",
    ano: "1994",
  },
  {
    marca: "Mercury",
    modelo: "Sable LS 3.0 V6",
    renavam: "50746089976",
    placa: "NEZ-3923",
    cor: "Preto",
    ano: "1992",
  },
  {
    marca: "LAMBORGHINI",
    modelo: "Gallardo Coupe Valentino Balboni LP550-2",
    renavam: "38018223190",
    placa: "JYY-5788",
    cor: "Amarelo",
    ano: "2010",
  },
  {
    marca: "Lexus",
    modelo: "LS 430 4.3 32V 281cv",
    renavam: "62506982220",
    placa: "NDP-1581",
    cor: "Cinza",
    ano: "2001",
  },
  {
    marca: "Buggy",
    modelo: "Buggy 1.6/ TST/ RS 1.6 4-Lug.",
    renavam: "22564302300",
    placa: "MYJ-8253",
    cor: "Prata",
    ano: "1985",
  },
  {
    marca: "Fibravan",
    modelo: "Buggy Off Road 1.8 8V",
    renavam: "36939814677",
    placa: "HWB-1241",
    cor: "Amarelo",
    ano: "2009",
  },
  {
    marca: "Agrale",
    modelo: "MARRU",
    renavam: "79389437837",
    placa: "MYV-5753",
    cor: "Azul",
    ano: "2008",
  },
  {
    marca: "Cross Lander",
    modelo: "CL-244 2.8 132cv 4x4 TB Int. Diesel",
    renavam: "57237910369",
    placa: "LWU-3781",
    cor: "Bege",
    ano: "2003",
  },
  {
    marca: "Mercury",
    modelo: "Sable LS 3.0 V6",
    renavam: "53254094113",
    placa: "LVV-7277",
    cor: "Dourado",
    ano: "1992",
  },
  {
    marca: "Ford",
    modelo: "Explorer XLT 4x2 4.0 V6",
    renavam: "88154751468",
    placa: "EKT-6176",
    cor: "Preto",
    ano: "1991",
  },
  {
    marca: "Fyber",
    modelo: "Buggy 2000W 1.8 8V/ 1.8 8V Flex",
    renavam: "43922298720",
    placa: "KHI-7680",
    cor: "Azul",
    ano: "2004",
  },
  {
    marca: "Alfa Romeo",
    modelo: "145 Quadrifoglio 2.0",
    renavam: "14242906222",
    placa: "MNC-7235",
    cor: "Prata",
    ano: "1996",
  },
  {
    marca: "Suzuki",
    modelo: "Swift Sport 1.6 16V 5p Mec.",
    renavam: "78758383378",
    placa: "CDZ-0098",
    cor: "Marrom",
    ano: "2015",
  },
  {
    marca: "Envemo",
    modelo: "Camper GL/GLS 4x4 4.0 Diesel",
    renavam: "35036583246",
    placa: "HTD-3589",
    cor: "Preto",
    ano: "1991",
  },
  {
    marca: "Seat",
    modelo: "Cordoba 1.6L",
    renavam: "87392555684",
    placa: "NEI-3513",
    cor: "Marrom",
    ano: "1999",
  },
  {
    marca: "Maserati",
    modelo: "GranCabrio 4.7 V8 32V 440cv",
    renavam: "12334605338",
    placa: "HZT-6782",
    cor: "Bege",
    ano: "2010",
  },
  {
    marca: "RELY",
    modelo: "PICK-UP Q22D CS 1.0 2p",
    renavam: "06347647579",
    placa: "NDC-6024",
    cor: "Azul",
    ano: "2014",
  },
  {
    marca: "Fyber",
    modelo: "Buggy 2000W 1.6 8V",
    renavam: "08889739094",
    placa: "HPY-1298",
    cor: "Marrom",
    ano: "2016",
  },
  {
    marca: "Rover",
    modelo: "Mini Cooper 1.3",
    renavam: "49835355767",
    placa: "JVO-4217",
    cor: "Preto",
    ano: "1992",
  },
  {
    marca: "RAM",
    modelo: "2500 LARAMIE SLT TROPIV. 6.7 4x4 Diesel",
    renavam: "53543910904",
    placa: "KCN-6547",
    cor: "Vermelho",
    ano: "2012",
  },
  {
    marca: "GEELY",
    modelo: "EC7 1.8 16V 130cv 4p Mec.",
    renavam: "58749677238",
    placa: "MRX-5895",
    cor: "Amarelo",
    ano: "2014",
  },
  {
    marca: "CHANA",
    modelo: "Cargo CE 1.0 8V 53cv (Pick-Up)",
    renavam: "57125267316",
    placa: "ITM-7657",
    cor: "Cinza",
    ano: "2006",
  },
  {
    marca: "Pontiac",
    modelo: "Trans-Sport SE 3.1",
    renavam: "66338019947",
    placa: "MVH-1522",
    cor: "Prata",
    ano: "1991",
  },
  {
    marca: "RAM",
    modelo: "2500 LARAMIE SLT TROPIV. 6.7 4x4 Diesel",
    renavam: "56405440800",
    placa: "MZW-3658",
    cor: "Verde",
    ano: "2012",
  },
];

function generateCompleteCar() {
  return cars[getRandomInt(0, cars.length - 1)];
}
function getRandomNumber(min, max) {
  return Math.floor(Math.random() * (max - min + 1)) + min;
}

// Função para gerar um endereço IP aleatório
function generateRandomIP() {
  const ipParts = [];
  for (let i = 0; i < 4; i++) {
    const part = getRandomNumber(0, 255);
    ipParts.push(part);
  }
  return ipParts.join(".");
}

const profissoes = [
  "Médico",
  "Engenheiro",
  "Professor",
  "Advogado",
  "Designer",
  "Programador",
  "Enfermeiro",
  "Contador",
  "Arquiteto",
  "Dentista",
  "Farmacêutico",
  "Psicólogo",
  "Economista",
  "Administrador",
  "Jornalista",
  "Policial",
  "Bombeiro",
  "Chef de Cozinha",
  "Piloto",
  "Ator",
  "Biólogo",
  "Geólogo",
  "Astrônomo",
  "Engenheiro Civil",
  "Veterinário",
  "Fisioterapeuta",
  "Nutricionista",
  "Eletricista",
  "Químico",
  "Matemático",
  "Bancário",
  "Fotógrafo",
  "Escritor",
  "Músico",
  "Artista Plástico",
  "Psiquiatra",
  "Engenheiro de Software",
  "Técnico de Informática",
  "Agricultor",
  "Carpinteiro",
  "Pintor",
  "Pescador",
  "Engenheiro Mecânico",
];
function generateProfession() {
  return profissoes[getRandomInt(0, profissoes.length - 1)];
}

module.exports = {
  generateProfession,
  generateRandomIP,
  generateNeighborhood,
  generateCEP,
  generateCNPJ,
  generateCity,
  generateStreet,
  generateState,
  generateCPF,
  generateEmails,
  generatePhone,
  generateRG,
  generateEnterprise,
  generateCNH,
  generateName,
  generatePISPASEP,
  generateCreditCard,
  generateAge,
  generateRNE,
  generatePlate,
  generateUuidv4,
  generateLoremIpsum,
  generateDate,
  generatePassword,
  generateFakePolicyNumber,
  generateCompleteCar,
};
